[
  {
    "id": 29001,
    "events": [
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 674611,
        "target_iid": 674611,
        "target_type": "Note",
        "author_id": 29001,
        "target_title": "GIMP 2.10.14 does not start on W10 (affects users using gimp-2.10.14-setup-2.exe on nVidia GPUs)",
        "created_at": "2019-12-29T00:08:09.187Z",
        "note": {
          "id": 674611,
          "type": null,
          "body": "Can also confirm this issue was happening with setup-2, when I switched to setup-1 the issue went away.\n\nOS: Win 10  \nGPU: RX 5700 XT  \nCPU: 3800X  \nAV Software: AVG",
          "attachment": null,
          "author": {
            "id": 29001,
            "name": "Xirema Xesirin",
            "username": "xiremaxesirin",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/3ed33fbf8cb44d05ffafc94123441a3c?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/xiremaxesirin"
          },
          "created_at": "2019-12-29T00:08:07.299Z",
          "updated_at": "2019-12-29T00:08:14.587Z",
          "system": false,
          "noteable_id": 77168,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4385
        },
        "author": {
          "id": 29001,
          "name": "Xirema Xesirin",
          "username": "xiremaxesirin",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/3ed33fbf8cb44d05ffafc94123441a3c?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/xiremaxesirin"
        },
        "author_username": "xiremaxesirin",
        "user_id": 29001
      }
    ]
  },
  {
    "id": 29002,
    "events": []
  },
  {
    "id": 29006,
    "events": [
      {
        "project_id": 9705,
        "action_name": "created",
        "target_id": null,
        "target_iid": null,
        "target_type": null,
        "author_id": 29006,
        "target_title": null,
        "created_at": "2019-12-29T05:00:21.657Z",
        "author": {
          "id": 29006,
          "name": "flameleo11",
          "username": "flameleo11",
          "state": "active",
          "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29006/avatar.png",
          "web_url": "https://gitlab.gnome.org/flameleo11"
        },
        "author_username": "flameleo11",
        "user_id": 29006
      }
    ]
  },
  {
    "id": 29007,
    "events": []
  },
  {
    "id": 29008,
    "events": [
      {
        "project_id": 546,
        "action_name": "commented on",
        "target_id": 674658,
        "target_iid": 674658,
        "target_type": "DiscussionNote",
        "author_id": 29008,
        "target_title": "MappingNotify event freezes drawing windows in Gnome for ~100ms",
        "created_at": "2019-12-29T08:36:08.558Z",
        "note": {
          "id": 674658,
          "type": "DiscussionNote",
          "body": "I'm working off popos (or Pop! OS), and I've been plagued with the same issue, but this fix worked for me. For some reason the init commands have never worked on here (might be something I did), but you don't need the first one, and replacing \"sudo init 5\" with just a restart works just as well. \n\nFantastic find Wigglyboy!",
          "attachment": null,
          "author": {
            "id": 29008,
            "name": "Gunther Huebler",
            "username": "guntherhuebler",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/b05bf631de826702e38e8e7ec2a7e360?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/guntherhuebler"
          },
          "created_at": "2019-12-29T08:36:07.397Z",
          "updated_at": "2019-12-29T08:36:16.667Z",
          "system": false,
          "noteable_id": 75026,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 1858
        },
        "author": {
          "id": 29008,
          "name": "Gunther Huebler",
          "username": "guntherhuebler",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/b05bf631de826702e38e8e7ec2a7e360?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/guntherhuebler"
        },
        "author_username": "guntherhuebler",
        "user_id": 29008
      }
    ]
  },
  {
    "id": 29009,
    "events": [
      {
        "project_id": 1534,
        "action_name": "opened",
        "target_id": 77487,
        "target_iid": 50,
        "target_type": "Issue",
        "author_id": 29009,
        "target_title": "Issue when setting an empty title ...",
        "created_at": "2019-12-29T09:53:20.235Z",
        "author": {
          "id": 29009,
          "name": "Jerome Kieffer",
          "username": "kif",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/8fe59ccb7b3ae314de742f6a715386e4?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/kif"
        },
        "author_username": "kif",
        "user_id": 29009
      },
      {
        "project_id": 1534,
        "action_name": "commented on",
        "target_id": 675378,
        "target_iid": 675378,
        "target_type": "Note",
        "author_id": 29009,
        "target_title": "Issue when setting an empty title ...",
        "created_at": "2019-12-31T08:49:33.324Z",
        "note": {
          "id": 675378,
          "type": null,
          "body": "I made it simple like (in ipython3):\n```\nIn [4]:  from gi.repository import GExiv2\n/usr/bin/ipython3:1: PyGIWarning: GExiv2 was imported without specifying a version first. Use gi.require_version('GExiv2', '0.10') before import to ensure that the right version gets loaded.\n  #! /bin/sh\nIn [6]: a = GExiv2.Metadata(\"15h58m30-NIKON_D7500.nef\")\n\nIn [7]: a.get_comment() #tag missing\n\nIn [8]: a.set_comment(\"\") #set empty value\n\nIn [9]: a.get_comment() #retrieve the next value \nOut[9]: 'lang=\"x-default\" '\n```",
          "attachment": null,
          "author": {
            "id": 29009,
            "name": "Jerome Kieffer",
            "username": "kif",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/8fe59ccb7b3ae314de742f6a715386e4?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/kif"
          },
          "created_at": "2019-12-31T08:49:32.757Z",
          "updated_at": "2019-12-31T08:49:32.757Z",
          "system": false,
          "noteable_id": 77487,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 50
        },
        "author": {
          "id": 29009,
          "name": "Jerome Kieffer",
          "username": "kif",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/8fe59ccb7b3ae314de742f6a715386e4?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/kif"
        },
        "author_username": "kif",
        "user_id": 29009
      }
    ]
  },
  {
    "id": 29010,
    "events": [
      {
        "project_id": 9706,
        "action_name": "created",
        "target_id": null,
        "target_iid": null,
        "target_type": null,
        "author_id": 29010,
        "target_title": null,
        "created_at": "2019-12-29T10:44:05.487Z",
        "author": {
          "id": 29010,
          "name": "amtech",
          "username": "amtech",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/0fd30b6ffe7564214c6f790f1746ab23?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/amtech"
        },
        "author_username": "amtech",
        "user_id": 29010
      }
    ]
  },
  {
    "id": 29011,
    "events": [
      {
        "project_id": 3560,
        "action_name": "opened",
        "target_id": 77490,
        "target_iid": 881,
        "target_type": "Issue",
        "author_id": 29011,
        "target_title": "paperwork-shell do not import a pdf",
        "created_at": "2019-12-29T10:53:27.082Z",
        "author": {
          "id": 29011,
          "name": "Brownie",
          "username": "Brownie2002",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/334f14797f0ed1971e8980a40e832fde?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/Brownie2002"
        },
        "author_username": "Brownie2002",
        "user_id": 29011
      },
      {
        "project_id": 3560,
        "action_name": "commented on",
        "target_id": 674715,
        "target_iid": 674715,
        "target_type": "Note",
        "author_id": 29011,
        "target_title": "paperwork-shell do not import a pdf",
        "created_at": "2019-12-29T13:50:17.239Z",
        "note": {
          "id": 674715,
          "type": null,
          "body": "Good guess ! I tested with a freshly created pdf, it is working like a charm ! I close the issue, sorry for the disturbance. Now, I have to focus on the scan ... My scanner is not reached with flatpack, but it is another story ! Thank you and have happy parties for the end of the year.",
          "attachment": null,
          "author": {
            "id": 29011,
            "name": "Brownie",
            "username": "Brownie2002",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/334f14797f0ed1971e8980a40e832fde?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/Brownie2002"
          },
          "created_at": "2019-12-29T13:50:16.969Z",
          "updated_at": "2019-12-29T13:50:16.969Z",
          "system": false,
          "noteable_id": 77490,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 881
        },
        "author": {
          "id": 29011,
          "name": "Brownie",
          "username": "Brownie2002",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/334f14797f0ed1971e8980a40e832fde?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/Brownie2002"
        },
        "author_username": "Brownie2002",
        "user_id": 29011
      },
      {
        "project_id": 3560,
        "action_name": "closed",
        "target_id": 77490,
        "target_iid": 881,
        "target_type": "Issue",
        "author_id": 29011,
        "target_title": "paperwork-shell do not import a pdf",
        "created_at": "2019-12-29T13:50:17.804Z",
        "author": {
          "id": 29011,
          "name": "Brownie",
          "username": "Brownie2002",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/334f14797f0ed1971e8980a40e832fde?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/Brownie2002"
        },
        "author_username": "Brownie2002",
        "user_id": 29011
      }
    ]
  },
  {
    "id": 29014,
    "events": [
      {
        "project_id": 1848,
        "action_name": "opened",
        "target_id": 77491,
        "target_iid": 4421,
        "target_type": "Issue",
        "author_id": 29014,
        "target_title": "Gimp crashed by closing project",
        "created_at": "2019-12-29T11:50:01.862Z",
        "author": {
          "id": 29014,
          "name": "Kornelius K.",
          "username": "korki43",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/29f000fd893ee67575ae1e108207dfb1?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/korki43"
        },
        "author_username": "korki43",
        "user_id": 29014
      },
      {
        "project_id": 1848,
        "action_name": "closed",
        "target_id": 77491,
        "target_iid": 4421,
        "target_type": "Issue",
        "author_id": 29014,
        "target_title": "Gimp crashed by closing project",
        "created_at": "2019-12-29T11:50:30.420Z",
        "author": {
          "id": 29014,
          "name": "Kornelius K.",
          "username": "korki43",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/29f000fd893ee67575ae1e108207dfb1?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/korki43"
        },
        "author_username": "korki43",
        "user_id": 29014
      },
      {
        "project_id": 1848,
        "action_name": "opened",
        "target_id": 77491,
        "target_iid": 4421,
        "target_type": "Issue",
        "author_id": 29014,
        "target_title": "Gimp crashed by closing project",
        "created_at": "2019-12-29T11:50:39.584Z",
        "author": {
          "id": 29014,
          "name": "Kornelius K.",
          "username": "korki43",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/29f000fd893ee67575ae1e108207dfb1?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/korki43"
        },
        "author_username": "korki43",
        "user_id": 29014
      },
      {
        "project_id": 1848,
        "action_name": "closed",
        "target_id": 77491,
        "target_iid": 4421,
        "target_type": "Issue",
        "author_id": 29014,
        "target_title": "Gimp crashed by closing project",
        "created_at": "2019-12-29T11:50:46.000Z",
        "author": {
          "id": 29014,
          "name": "Kornelius K.",
          "username": "korki43",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/29f000fd893ee67575ae1e108207dfb1?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/korki43"
        },
        "author_username": "korki43",
        "user_id": 29014
      },
      {
        "project_id": 1848,
        "action_name": "opened",
        "target_id": 77491,
        "target_iid": 4421,
        "target_type": "Issue",
        "author_id": 29014,
        "target_title": "Gimp crashed by closing project",
        "created_at": "2019-12-29T11:50:49.853Z",
        "author": {
          "id": 29014,
          "name": "Kornelius K.",
          "username": "korki43",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/29f000fd893ee67575ae1e108207dfb1?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/korki43"
        },
        "author_username": "korki43",
        "user_id": 29014
      }
    ]
  },
  {
    "id": 29015,
    "events": [
      {
        "project_id": 558,
        "action_name": "commented on",
        "target_id": 674706,
        "target_iid": 674706,
        "target_type": "Note",
        "author_id": 29015,
        "target_title": "Unable to download updates from \"extensions.gnome.org\" [...]: Cannot resolve hostname",
        "created_at": "2019-12-29T11:58:20.487Z",
        "note": {
          "id": 674706,
          "type": null,
          "body": "Same issue on debian 10 buster with gnome wayland. :'(![proof](/uploads/aa875ce06af777981cefffd192257607/proof.png)",
          "attachment": null,
          "author": {
            "id": 29015,
            "name": "Enguerrand ARMINJON",
            "username": "EArminjon",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/325d0f3637f49f5e5111e3b5a921dcdd?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/EArminjon"
          },
          "created_at": "2019-12-29T11:58:19.598Z",
          "updated_at": "2019-12-29T11:58:38.248Z",
          "system": false,
          "noteable_id": 67055,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 694
        },
        "author": {
          "id": 29015,
          "name": "Enguerrand ARMINJON",
          "username": "EArminjon",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/325d0f3637f49f5e5111e3b5a921dcdd?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/EArminjon"
        },
        "author_username": "EArminjon",
        "user_id": 29015
      }
    ]
  },
  {
    "id": 29016,
    "events": [
      {
        "project_id": 660,
        "action_name": "commented on",
        "target_id": 674709,
        "target_iid": 674709,
        "target_type": "Note",
        "author_id": 29016,
        "target_title": "let user choose one bluetooth device from several in gnome control center",
        "created_at": "2019-12-29T12:31:23.708Z",
        "note": {
          "id": 674709,
          "type": null,
          "body": "Would love to see this feature.\nThe vive lightouse dongle is killing me...",
          "attachment": null,
          "author": {
            "id": 29016,
            "name": "extraymond",
            "username": "extraymond",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/3a5b2f9b5913e6a17f2930315b84f35a?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/extraymond"
          },
          "created_at": "2019-12-29T12:31:22.976Z",
          "updated_at": "2019-12-29T12:31:22.976Z",
          "system": false,
          "noteable_id": 54202,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 263
        },
        "author": {
          "id": 29016,
          "name": "extraymond",
          "username": "extraymond",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/3a5b2f9b5913e6a17f2930315b84f35a?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/extraymond"
        },
        "author_username": "extraymond",
        "user_id": 29016
      }
    ]
  },
  {
    "id": 29019,
    "events": [
      {
        "project_id": 1624,
        "action_name": "commented on",
        "target_id": 674732,
        "target_iid": 674732,
        "target_type": "Note",
        "author_id": 29019,
        "target_title": "Add support for Flatpak packages",
        "created_at": "2019-12-29T15:35:25.253Z",
        "note": {
          "id": 674732,
          "type": null,
          "body": "Any progress ?",
          "attachment": null,
          "author": {
            "id": 29019,
            "name": "Stavpup",
            "username": "stavpup",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/c2c34ffd580bf87485847f6eda5a9235?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/stavpup"
          },
          "created_at": "2019-12-29T15:35:24.563Z",
          "updated_at": "2019-12-29T15:35:24.563Z",
          "system": false,
          "noteable_id": 48752,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 13
        },
        "author": {
          "id": 29019,
          "name": "Stavpup",
          "username": "stavpup",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/c2c34ffd580bf87485847f6eda5a9235?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/stavpup"
        },
        "author_username": "stavpup",
        "user_id": 29019
      }
    ]
  },
  {
    "id": 29020,
    "events": [
      {
        "project_id": 546,
        "action_name": "commented on",
        "target_id": 674739,
        "target_iid": 674739,
        "target_type": "Note",
        "author_id": 29020,
        "target_title": "Update Gnome Shell extensions from the command line",
        "created_at": "2019-12-29T16:26:33.560Z",
        "note": {
          "id": 674739,
          "type": null,
          "body": "Couldn't `org.gnome.Shell.Extensions.CheckForUpdates` be used for this (defined [here](https://github.com/GNOME/gnome-shell/blame/ebf77748a8477eb712287273b4b92a8e9a189e22/data/dbus-interfaces/org.gnome.Shell.Extensions.xml#L214) and implemented [here](https://github.com/GNOME/gnome-shell/blame/ebf77748a8477eb712287273b4b92a8e9a189e22/js/ui/extensionDownloader.js#L154))?\n```\ndbus-send --session --print-reply --type=method_call --dest=org.gnome.Shell /org/gnome/Shell org.gnome.Shell.Extensions.CheckForUpdates\n```\nUnfortunately, this fails on my GNOME Shell 3.34.2 with the following error:\n```\ngnome-shell[4608]: JS ERROR: Exception in method call: CheckForUpdates: TypeError: Main.extensionManager.extensions is undefined\ncheckForUpdates/<@resource:///org/gnome/shell/ui/extensionDownloader.js:157:9\ncheckForUpdates@resource:///org/gnome/shell/ui/extensionDownloader.js:156:5\nCheckForUpdates@resource:///org/gnome/shell/ui/shellDBus.js:320:9\n_handleMethodCall@resource:///org/gnome/gjs/modules/overrides/Gio.js:327:67\n_wrapJSObject/<@resource:///org/gnome/gjs/modules/overrides/Gio.js:404:16\n ```\nCurrently, I'm using [this script](https://gist.github.com/N-Coder/9ec0eb11c2f14013c112e1468c708b30) (which is heavily inspired by the implementation of the \"chrome-gnome-shell\" daemon) to automatically click all the green update buttons and then disable unused extensions again. Of course, this probably needs a shell restart afterwards, but I usually don't want to quit all my applications (which seems to be a new thing under wayland) just for that.",
          "attachment": null,
          "author": {
            "id": 29020,
            "name": "Niko Fink",
            "username": "N-Coder",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/eca9cbb4abd13d1798da87a7c8b9cc09?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/N-Coder"
          },
          "created_at": "2019-12-29T16:26:32.429Z",
          "updated_at": "2019-12-29T16:26:32.429Z",
          "system": false,
          "noteable_id": 57492,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 906
        },
        "author": {
          "id": 29020,
          "name": "Niko Fink",
          "username": "N-Coder",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/eca9cbb4abd13d1798da87a7c8b9cc09?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/N-Coder"
        },
        "author_username": "N-Coder",
        "user_id": 29020
      }
    ]
  },
  {
    "id": 29022,
    "events": [
      {
        "project_id": 1856,
        "action_name": "commented on",
        "target_id": 674794,
        "target_iid": 674794,
        "target_type": "Note",
        "author_id": 29022,
        "target_title": "gnome-session[823]: WARNING: Could not parse desktop file orca-autostart.desktop or it references a not found TryExec binary",
        "created_at": "2019-12-29T20:07:12.711Z",
        "note": {
          "id": 674794,
          "type": null,
          "body": "Having spurious warnings in logs makes debugging other (real) issues more complicated.",
          "attachment": null,
          "author": {
            "id": 29022,
            "name": "Marti Raudsepp",
            "username": "intgr",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/485b471504b5332261dcf8e67b068b04?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/intgr"
          },
          "created_at": "2019-12-29T20:07:12.137Z",
          "updated_at": "2019-12-29T20:07:12.137Z",
          "system": false,
          "noteable_id": 36617,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 252
        },
        "author": {
          "id": 29022,
          "name": "Marti Raudsepp",
          "username": "intgr",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/485b471504b5332261dcf8e67b068b04?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/intgr"
        },
        "author_username": "intgr",
        "user_id": 29022
      },
      {
        "project_id": 1856,
        "action_name": "commented on",
        "target_id": 674837,
        "target_iid": 674837,
        "target_type": "Note",
        "author_id": 29022,
        "target_title": "Please watch the CanGraphical property on logind Seat objects before spawning a graphical login on a seat",
        "created_at": "2019-12-29T22:02:46.255Z",
        "note": {
          "id": 674837,
          "type": null,
          "body": "I've been hitting this bug for many months now on Arch Linux. Took me a while to get around to debugging this. Meanwhile many Arch Linux users are perpetrating [the work-around to bake KMS drivers into initramfs](https://wiki.archlinux.org/index.php/Kernel_mode_setting#Early_KMS_start), but that's not a proper solution.\n\nLogs: https://gist.github.com/intgr/a7eee474994e8a845405a7466ac2ebc2\n\nPlease fix, it's really disappointing to see design flaws that were known about 7 years ago bite users.\n\n(Issue https://github.com/systemd/systemd/issues/10435 helped me in finding this)",
          "attachment": null,
          "author": {
            "id": 29022,
            "name": "Marti Raudsepp",
            "username": "intgr",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/485b471504b5332261dcf8e67b068b04?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/intgr"
          },
          "created_at": "2019-12-29T22:02:45.359Z",
          "updated_at": "2019-12-29T22:02:45.359Z",
          "system": false,
          "noteable_id": 36070,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 103
        },
        "author": {
          "id": 29022,
          "name": "Marti Raudsepp",
          "username": "intgr",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/485b471504b5332261dcf8e67b068b04?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/intgr"
        },
        "author_username": "intgr",
        "user_id": 29022
      },
      {
        "project_id": 1856,
        "action_name": "commented on",
        "target_id": 674858,
        "target_iid": 674858,
        "target_type": "Note",
        "author_id": 29022,
        "target_title": "Please watch the CanGraphical property on logind Seat objects before spawning a graphical login on a seat",
        "created_at": "2019-12-29T22:54:49.773Z",
        "note": {
          "id": 674858,
          "type": null,
          "body": "Thanks @mraudsepp, that's very helpful and solves my issue.\n\nI created an Arch Linux bug report to integrate this patch: https://bugs.archlinux.org/task/64985",
          "attachment": null,
          "author": {
            "id": 29022,
            "name": "Marti Raudsepp",
            "username": "intgr",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/485b471504b5332261dcf8e67b068b04?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/intgr"
          },
          "created_at": "2019-12-29T22:54:47.224Z",
          "updated_at": "2019-12-29T22:54:47.224Z",
          "system": false,
          "noteable_id": 36070,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 103
        },
        "author": {
          "id": 29022,
          "name": "Marti Raudsepp",
          "username": "intgr",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/485b471504b5332261dcf8e67b068b04?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/intgr"
        },
        "author_username": "intgr",
        "user_id": 29022
      }
    ]
  },
  {
    "id": 29023,
    "events": [
      {
        "project_id": 416,
        "action_name": "opened",
        "target_id": 77504,
        "target_iid": 375,
        "target_type": "Issue",
        "author_id": 29023,
        "target_title": "```from gi.repository import Gtk``` fails under python 3.8",
        "created_at": "2019-12-29T21:08:58.425Z",
        "author": {
          "id": 29023,
          "name": "Yoram Zarai",
          "username": "yoram.zarai",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/88bf86ed5ece41f5ab19173b003d5698?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/yoram.zarai"
        },
        "author_username": "yoram.zarai",
        "user_id": 29023
      },
      {
        "project_id": 416,
        "action_name": "commented on",
        "target_id": 677127,
        "target_iid": 677127,
        "target_type": "Note",
        "author_id": 29023,
        "target_title": "```from gi.repository import Gtk``` fails under python 3.8",
        "created_at": "2020-01-04T20:04:04.922Z",
        "note": {
          "id": 677127,
          "type": null,
          "body": "Hi. Thank you for the reply. I am not using brew but macports. I have never tested that before. It came up from an issue that I have with[TexText under Inkscape](https://github.com/textext/textext/issues/150). I tried it as well using Python 3.7 (and using the port py37-gobject3) and in that case after executing `from gi.repository import Gtk` I don't get the prompt back (seems to get stuck).",
          "attachment": null,
          "author": {
            "id": 29023,
            "name": "Yoram Zarai",
            "username": "yoram.zarai",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/88bf86ed5ece41f5ab19173b003d5698?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/yoram.zarai"
          },
          "created_at": "2020-01-04T20:04:04.451Z",
          "updated_at": "2020-01-04T20:04:04.451Z",
          "system": false,
          "noteable_id": 77504,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 375
        },
        "author": {
          "id": 29023,
          "name": "Yoram Zarai",
          "username": "yoram.zarai",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/88bf86ed5ece41f5ab19173b003d5698?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/yoram.zarai"
        },
        "author_username": "yoram.zarai",
        "user_id": 29023
      }
    ]
  },
  {
    "id": 29025,
    "events": [
      {
        "project_id": 1848,
        "action_name": "opened",
        "target_id": 77505,
        "target_iid": 4424,
        "target_type": "Issue",
        "author_id": 29025,
        "target_title": "GIMP crashed with a fatal error: fatal error: unhandled exception",
        "created_at": "2019-12-29T22:06:26.981Z",
        "author": {
          "id": 29025,
          "name": "Dan Barnes.",
          "username": "11below",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/513ee772260ff38646e2296aab5ccb82?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/11below"
        },
        "author_username": "11below",
        "user_id": 29025
      }
    ]
  },
  {
    "id": 29026,
    "events": [
      {
        "project_id": 9707,
        "action_name": "created",
        "target_id": null,
        "target_iid": null,
        "target_type": null,
        "author_id": 29026,
        "target_title": null,
        "created_at": "2019-12-29T22:08:37.279Z",
        "author": {
          "id": 29026,
          "name": "Fr\u00e9d\u00e9ric Fauberteau",
          "username": "triaxx",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/6edc7edff0671d490ad5213ce32a5f61?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/triaxx"
        },
        "author_username": "triaxx",
        "user_id": 29026
      },
      {
        "project_id": 9707,
        "action_name": "pushed new",
        "target_id": null,
        "target_iid": null,
        "target_type": null,
        "author_id": 29026,
        "target_title": null,
        "created_at": "2019-12-29T22:17:10.823Z",
        "author": {
          "id": 29026,
          "name": "Fr\u00e9d\u00e9ric Fauberteau",
          "username": "triaxx",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/6edc7edff0671d490ad5213ce32a5f61?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/triaxx"
        },
        "push_data": {
          "commit_count": 1,
          "action": "created",
          "ref_type": "branch",
          "commit_from": null,
          "commit_to": "65f12407bf58bc59525fa6d5b428c22b5ba80200",
          "ref": "triaxx-master-patch-78526",
          "commit_title": "Fix build error with FreeBSD",
          "ref_count": null
        },
        "author_username": "triaxx",
        "user_id": 29026
      },
      {
        "project_id": 658,
        "action_name": "opened",
        "target_id": 21199,
        "target_iid": 1306,
        "target_type": "MergeRequest",
        "author_id": 29026,
        "target_title": "Fix build error with FreeBSD",
        "created_at": "2019-12-30T05:55:50.635Z",
        "author": {
          "id": 29026,
          "name": "Fr\u00e9d\u00e9ric Fauberteau",
          "username": "triaxx",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/6edc7edff0671d490ad5213ce32a5f61?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/triaxx"
        },
        "author_username": "triaxx",
        "user_id": 29026
      },
      {
        "project_id": 658,
        "action_name": "commented on",
        "target_id": 677768,
        "target_iid": 677768,
        "target_type": "Note",
        "author_id": 29026,
        "target_title": "Fix build error with FreeBSD",
        "created_at": "2020-01-05T23:40:54.841Z",
        "note": {
          "id": 677768,
          "type": null,
          "body": "In order to provide some context, I use the pkgsrc package framework, I use FreeBSD 12.0-RELEASE-p12 amd64, the linker is LLD 6.0.1 (FreeBSD 335540-1200005) (compatible with GNU linkers) and the error is:\n\n```\nclang  -o glib/libglib-2.0.so.0.6200.3 'glib/4430778@@glib-2.0@sha/deprecated_gallocator.c.o' 'glib/4430778@@glib-2.0@sha/deprecated_gcache.c.o' 'glib/4430778@@glib-2.0@sha/deprecated_gcompletion.c.o' 'glib/4430778@@glib-2.0@sha/deprecated_grel.c.o' 'glib/4430778@@glib-2.0@sha/deprecated_gthread-deprecated.c.o' 'glib/4430778@@glib-2.0@sha/garcbox.c.o' 'glib/4430778@@glib-2.0@sha/garray.c.o' 'glib/4430778@@glib-2.0@sha/gasyncqueue.c.o' 'glib/4430778@@glib-2.0@sha/gatomic.c.o' 'glib/4430778@@glib-2.0@sha/gbacktrace.c.o' 'glib/4430778@@glib-2.0@sha/gbase64.c.o' 'glib/4430778@@glib-2.0@sha/gbitlock.c.o' 'glib/4430778@@glib-2.0@sha/gbookmarkfile.c.o' 'glib/4430778@@glib-2.0@sha/gbytes.c.o' 'glib/4430778@@glib-2.0@sha/gcharset.c.o' 'glib/4430778@@glib-2.0@sha/gchecksum.c.o' 'glib/4430778@@glib-2.0@sha/gconvert.c.o' 'glib/4430778@@glib-2.0@sha/gdataset.c.o' 'glib/4430778@@glib-2.0@sha/gdate.c.o' 'glib/4430778@@glib-2.0@sha/gdatetime.c.o' 'glib/4430778@@glib-2.0@sha/gdir.c.o' 'glib/4430778@@glib-2.0@sha/genviron.c.o' 'glib/4430778@@glib-2.0@sha/gerror.c.o' 'glib/4430778@@glib-2.0@sha/gfileutils.c.o' 'glib/4430778@@glib-2.0@sha/ggettext.c.o' 'glib/4430778@@glib-2.0@sha/ghash.c.o' 'glib/4430778@@glib-2.0@sha/ghmac.c.o' 'glib/4430778@@glib-2.0@sha/ghook.c.o' 'glib/4430778@@glib-2.0@sha/ghostutils.c.o' 'glib/4430778@@glib-2.0@sha/giochannel.c.o' 'glib/4430778@@glib-2.0@sha/gkeyfile.c.o' 'glib/4430778@@glib-2.0@sha/glib-init.c.o' 'glib/4430778@@glib-2.0@sha/glib-private.c.o' 'glib/4430778@@glib-2.0@sha/glist.c.o' 'glib/4430778@@glib-2.0@sha/gmain.c.o' 'glib/4430778@@glib-2.0@sha/gmappedfile.c.o' 'glib/4430778@@glib-2.0@sha/gmarkup.c.o' 'glib/4430778@@glib-2.0@sha/gmem.c.o' 'glib/4430778@@glib-2.0@sha/gmessages.c.o' 'glib/4430778@@glib-2.0@sha/gnode.c.o' 'glib/4430778@@glib-2.0@sha/goption.c.o' 'glib/4430778@@glib-2.0@sha/gpattern.c.o' 'glib/4430778@@glib-2.0@sha/gpoll.c.o' 'glib/4430778@@glib-2.0@sha/gprimes.c.o' 'glib/4430778@@glib-2.0@sha/gqsort.c.o' 'glib/4430778@@glib-2.0@sha/gquark.c.o' 'glib/4430778@@glib-2.0@sha/gqueue.c.o' 'glib/4430778@@glib-2.0@sha/grand.c.o' 'glib/4430778@@glib-2.0@sha/grcbox.c.o' 'glib/4430778@@glib-2.0@sha/grefcount.c.o' 'glib/4430778@@glib-2.0@sha/grefstring.c.o' 'glib/4430778@@glib-2.0@sha/gregex.c.o' 'glib/4430778@@glib-2.0@sha/gscanner.c.o' 'glib/4430778@@glib-2.0@sha/gsequence.c.o' 'glib/4430778@@glib-2.0@sha/gshell.c.o' 'glib/4430778@@glib-2.0@sha/gslice.c.o' 'glib/4430778@@glib-2.0@sha/gslist.c.o' 'glib/4430778@@glib-2.0@sha/gstdio.c.o' 'glib/4430778@@glib-2.0@sha/gstrfuncs.c.o' 'glib/4430778@@glib-2.0@sha/gstring.c.o' 'glib/4430778@@glib-2.0@sha/gstringchunk.c.o' 'glib/4430778@@glib-2.0@sha/gtestutils.c.o' 'glib/4430778@@glib-2.0@sha/gthread.c.o' 'glib/4430778@@glib-2.0@sha/gthreadpool.c.o' 'glib/4430778@@glib-2.0@sha/gtimer.c.o' 'glib/4430778@@glib-2.0@sha/gtimezone.c.o' 'glib/4430778@@glib-2.0@sha/gtranslit.c.o' 'glib/4430778@@glib-2.0@sha/gtrashstack.c.o' 'glib/4430778@@glib-2.0@sha/gtree.c.o' 'glib/4430778@@glib-2.0@sha/guniprop.c.o' 'glib/4430778@@glib-2.0@sha/gutf8.c.o' 'glib/4430778@@glib-2.0@sha/gunibreak.c.o' 'glib/4430778@@glib-2.0@sha/gunicollate.c.o' 'glib/4430778@@glib-2.0@sha/gunidecomp.c.o' 'glib/4430778@@glib-2.0@sha/gurifuncs.c.o' 'glib/4430778@@glib-2.0@sha/gutils.c.o' 'glib/4430778@@glib-2.0@sha/guuid.c.o' 'glib/4430778@@glib-2.0@sha/gvariant.c.o' 'glib/4430778@@glib-2.0@sha/gvariant-core.c.o' 'glib/4430778@@glib-2.0@sha/gvariant-parser.c.o' 'glib/4430778@@glib-2.0@sha/gvariant-serialiser.c.o' 'glib/4430778@@glib-2.0@sha/gvarianttypeinfo.c.o' 'glib/4430778@@glib-2.0@sha/gvarianttype.c.o' 'glib/4430778@@glib-2.0@sha/gversion.c.o' 'glib/4430778@@glib-2.0@sha/gwakeup.c.o' 'glib/4430778@@glib-2.0@sha/gprintf.c.o' 'glib/4430778@@glib-2.0@sha/glib-unix.c.o' 'glib/4430778@@glib-2.0@sha/gspawn.c.o' 'glib/4430778@@glib-2.0@sha/giounix.c.o' 'glib/4430778@@glib-2.0@sha/gthread-posix.c.o' 'glib/libcharset/6e4c96c@@charset@sta/localcharset.c.o' -I/usr/include -I/usr/pkg/include -I/usr/pkg/include/python3.8 -L/usr/lib -L/usr/pkg/lib -Wl,--as-needed -Wl,--no-undefined -shared -fPIC -Wl,--start-group -Wl,-soname,libglib-2.0.so.0 -O2 -Wl,-R/usr/lib -Wl,-R/usr/pkg/lib -Wl,-z,nodelete -Wl,-Bsymbolic-functions -Wl,-R/usr/pkg/lib /usr/pkg/lib/libpcre.so -pthread -lintl -Wl,--end-group\n/usr/bin/ld: error: undefined symbol: environ\n>>> referenced by genviron.c\n>>>               glib/4430778@@glib-2.0@sha/genviron.c.o:(g_listenv)\n\n/usr/bin/ld: error: undefined symbol: environ\n>>> referenced by genviron.c\n>>>               glib/4430778@@glib-2.0@sha/genviron.c.o:(g_listenv)\n\n/usr/bin/ld: error: undefined symbol: environ\n>>> referenced by genviron.c\n>>>               glib/4430778@@glib-2.0@sha/genviron.c.o:(g_get_environ)\n\n/usr/bin/ld: error: undefined symbol: environ\n>>> referenced by gspawn.c\n>>>               glib/4430778@@glib-2.0@sha/gspawn.c.o:(fork_exec_with_fds)\nclang: error: linker command failed with exit code 1 (use -v to see invocation)\nninja: build stopped: subcommand failed.\n*** Error code 1\n\nStop.\nbmake[1]: stopped in /usr/pkgsrc/devel/glib2\n*** Error code 1\n\nStop.\n```\nActually, like said by @lantw, the problems with environ are known on FreeBSD: [devel/glib20: ld-elf.so.1: /usr/local/lib/libglib-2.0.so.0: Undefined symbol \"environ\" (WITH_LLD_IS_LD)](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=220103)\nI tested to add extern char **environ but it does not seem to be sufficient to solve the problem.\nI initially passed -Db_lundef=false to meson but own suggest to me to make environ a \"weak\" symbol.",
          "attachment": null,
          "author": {
            "id": 29026,
            "name": "Fr\u00e9d\u00e9ric Fauberteau",
            "username": "triaxx",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/6edc7edff0671d490ad5213ce32a5f61?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/triaxx"
          },
          "created_at": "2020-01-05T23:40:54.065Z",
          "updated_at": "2020-01-05T23:40:54.065Z",
          "system": false,
          "noteable_id": 21199,
          "noteable_type": "MergeRequest",
          "resolvable": false,
          "noteable_iid": 1306
        },
        "author": {
          "id": 29026,
          "name": "Fr\u00e9d\u00e9ric Fauberteau",
          "username": "triaxx",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/6edc7edff0671d490ad5213ce32a5f61?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/triaxx"
        },
        "author_username": "triaxx",
        "user_id": 29026
      },
      {
        "project_id": 658,
        "action_name": "closed",
        "target_id": 21199,
        "target_iid": 1306,
        "target_type": "MergeRequest",
        "author_id": 29026,
        "target_title": "Fix build error with FreeBSD",
        "created_at": "2020-01-18T08:20:23.175Z",
        "author": {
          "id": 29026,
          "name": "Fr\u00e9d\u00e9ric Fauberteau",
          "username": "triaxx",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/6edc7edff0671d490ad5213ce32a5f61?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/triaxx"
        },
        "author_username": "triaxx",
        "user_id": 29026
      },
      {
        "project_id": 658,
        "action_name": "commented on",
        "target_id": 688431,
        "target_iid": 688431,
        "target_type": "Note",
        "author_id": 29026,
        "target_title": "Fix build error with FreeBSD",
        "created_at": "2020-01-18T08:20:23.238Z",
        "note": {
          "id": 688431,
          "type": null,
          "body": "Maybe because I use the pkgsrc framework to build glib and you don't.\n\nMaybe it works with /usr/bin/ld.bfd but I think it is easier for us to have a small patch for this package than hack it to use a different linker.\n\nMaybe this bug is an artifact and you can safely ignore my merge request.",
          "attachment": null,
          "author": {
            "id": 29026,
            "name": "Fr\u00e9d\u00e9ric Fauberteau",
            "username": "triaxx",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/6edc7edff0671d490ad5213ce32a5f61?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/triaxx"
          },
          "created_at": "2020-01-18T08:20:22.179Z",
          "updated_at": "2020-01-18T08:20:22.179Z",
          "system": false,
          "noteable_id": 21199,
          "noteable_type": "MergeRequest",
          "resolvable": false,
          "noteable_iid": 1306
        },
        "author": {
          "id": 29026,
          "name": "Fr\u00e9d\u00e9ric Fauberteau",
          "username": "triaxx",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/6edc7edff0671d490ad5213ce32a5f61?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/triaxx"
        },
        "author_username": "triaxx",
        "user_id": 29026
      },
      {
        "project_id": 658,
        "action_name": "commented on",
        "target_id": 689275,
        "target_iid": 689275,
        "target_type": "DiscussionNote",
        "author_id": 29026,
        "target_title": "Fix build error with FreeBSD",
        "created_at": "2020-01-19T20:27:46.141Z",
        "note": {
          "id": 689275,
          "type": "DiscussionNote",
          "body": ">  FreeBSD CI machine uses `-Db_lundef=false`.\n\nCould it be added to the meson configuration file please?",
          "attachment": null,
          "author": {
            "id": 29026,
            "name": "Fr\u00e9d\u00e9ric Fauberteau",
            "username": "triaxx",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/6edc7edff0671d490ad5213ce32a5f61?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/triaxx"
          },
          "created_at": "2020-01-19T20:27:45.372Z",
          "updated_at": "2020-01-19T20:27:52.948Z",
          "system": false,
          "noteable_id": 21199,
          "noteable_type": "MergeRequest",
          "resolvable": true,
          "resolved": true,
          "resolved_by": {
            "id": 421,
            "name": "Philip Withnall",
            "username": "pwithnall",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/31cf842413ffa02fa5a7d8f105e2e7c9?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/pwithnall"
          },
          "noteable_iid": 1306
        },
        "author": {
          "id": 29026,
          "name": "Fr\u00e9d\u00e9ric Fauberteau",
          "username": "triaxx",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/6edc7edff0671d490ad5213ce32a5f61?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/triaxx"
        },
        "author_username": "triaxx",
        "user_id": 29026
      }
    ]
  },
  {
    "id": 29027,
    "events": [
      {
        "project_id": 1848,
        "action_name": "opened",
        "target_id": 77513,
        "target_iid": 4425,
        "target_type": "Issue",
        "author_id": 29027,
        "target_title": "Saving to PDF changes text's font and size",
        "created_at": "2019-12-30T03:08:22.750Z",
        "author": {
          "id": 29027,
          "name": "Christine Gardner",
          "username": "gardnerchristine52",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/gardnerchristine52"
        },
        "author_username": "gardnerchristine52",
        "user_id": 29027
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 674911,
        "target_iid": 674911,
        "target_type": "Note",
        "author_id": 29027,
        "target_title": "Saving to PDF changes text's font and size",
        "created_at": "2019-12-30T04:49:18.939Z",
        "note": {
          "id": 674911,
          "type": null,
          "body": "It is GIMP 2.10.14 but it seems a bit different to the version on my old computer, which has the same numbers. I don't understand the Linux reference but I use Windows if that's relevant. \n\nI've tried a few fonts and it seems to make no difference. I don't know what Type 1 fonts are. The image I'm using for this test is a new one and everything seems fine until I try to save to pdf. \n\nThanks Liam.[STAIRWAYTOHELL.xcf](/uploads/7a1e768b5448c1bd5ab4506a65f27f18/STAIRWAYTOHELL.xcf)[BENSTAIRWAYTOHELL.pdf](/uploads/057b94a33e481030842b314adff89fda/BENSTAIRWAYTOHELL.pdf)![BENSTAIRWAYTOHELL](/uploads/f6ba28ae387f93351e81473bfdbb197c/BENSTAIRWAYTOHELL.jpg)[BENSTAIRWAYTOHELL.xcf](/uploads/52a224d292a9ae8b9a8c60316e9b51a3/BENSTAIRWAYTOHELL.xcf)",
          "attachment": null,
          "author": {
            "id": 29027,
            "name": "Christine Gardner",
            "username": "gardnerchristine52",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/gardnerchristine52"
          },
          "created_at": "2019-12-30T04:49:18.276Z",
          "updated_at": "2019-12-30T04:49:18.276Z",
          "system": false,
          "noteable_id": 77513,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4425
        },
        "author": {
          "id": 29027,
          "name": "Christine Gardner",
          "username": "gardnerchristine52",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/gardnerchristine52"
        },
        "author_username": "gardnerchristine52",
        "user_id": 29027
      },
      {
        "project_id": 1848,
        "action_name": "closed",
        "target_id": 77513,
        "target_iid": 4425,
        "target_type": "Issue",
        "author_id": 29027,
        "target_title": "Saving to PDF changes text's font and size",
        "created_at": "2019-12-30T04:49:19.362Z",
        "author": {
          "id": 29027,
          "name": "Christine Gardner",
          "username": "gardnerchristine52",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/gardnerchristine52"
        },
        "author_username": "gardnerchristine52",
        "user_id": 29027
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 674927,
        "target_iid": 674927,
        "target_type": "Note",
        "author_id": 29027,
        "target_title": "Saving to PDF changes text's font and size",
        "created_at": "2019-12-30T07:54:43.251Z",
        "note": {
          "id": 674927,
          "type": null,
          "body": "No. I apparently did something wrong and my message didn't go through. I'll\ntry again tomorrow. Thanks.",
          "attachment": null,
          "author": {
            "id": 29027,
            "name": "Christine Gardner",
            "username": "gardnerchristine52",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/gardnerchristine52"
          },
          "created_at": "2019-12-30T07:54:42.377Z",
          "updated_at": "2019-12-30T07:54:42.377Z",
          "system": false,
          "noteable_id": 77513,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4425
        },
        "author": {
          "id": 29027,
          "name": "Christine Gardner",
          "username": "gardnerchristine52",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/gardnerchristine52"
        },
        "author_username": "gardnerchristine52",
        "user_id": 29027
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 676292,
        "target_iid": 676292,
        "target_type": "Note",
        "author_id": 29027,
        "target_title": "Saving to PDF changes text's font and size",
        "created_at": "2020-01-03T04:23:17.426Z",
        "note": {
          "id": 676292,
          "type": null,
          "body": "I'm a bit lost with all the tech talk but is there any solution to\nmy problem? I'm still unable to convert certain fonts to pdf with GIMP,\nalthough the earlier version on my old computer still does it just fine.",
          "attachment": null,
          "author": {
            "id": 29027,
            "name": "Christine Gardner",
            "username": "gardnerchristine52",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/gardnerchristine52"
          },
          "created_at": "2020-01-03T04:23:16.702Z",
          "updated_at": "2020-01-03T04:23:16.702Z",
          "system": false,
          "noteable_id": 77513,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4425
        },
        "author": {
          "id": 29027,
          "name": "Christine Gardner",
          "username": "gardnerchristine52",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/gardnerchristine52"
        },
        "author_username": "gardnerchristine52",
        "user_id": 29027
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 676448,
        "target_iid": 676448,
        "target_type": "Note",
        "author_id": 29027,
        "target_title": "Saving to PDF changes text's font and size",
        "created_at": "2020-01-03T12:10:32.842Z",
        "note": {
          "id": 676448,
          "type": null,
          "body": "One of the fonts, Euphorigenec, was on an xcf file from my old computer. The new one recognised and installed it. The other one, Bene something, was already on the new GIMP.",
          "attachment": null,
          "author": {
            "id": 29027,
            "name": "Christine Gardner",
            "username": "gardnerchristine52",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/gardnerchristine52"
          },
          "created_at": "2020-01-03T12:10:32.038Z",
          "updated_at": "2020-01-03T12:10:32.038Z",
          "system": false,
          "noteable_id": 77513,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4425
        },
        "author": {
          "id": 29027,
          "name": "Christine Gardner",
          "username": "gardnerchristine52",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/gardnerchristine52"
        },
        "author_username": "gardnerchristine52",
        "user_id": 29027
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 679594,
        "target_iid": 679594,
        "target_type": "DiscussionNote",
        "author_id": 29027,
        "target_title": "Saving to PDF changes text's font and size",
        "created_at": "2020-01-07T20:25:27.770Z",
        "note": {
          "id": 679594,
          "type": "DiscussionNote",
          "body": "I'm not sure if everyone here is still talking about my issue with the fonts not converting correctly to pdf. My son had a look and said that the older version of GIMP automatically flattened the layers but the new doesn't do that. I flattened them myself and all fonts are fine.",
          "attachment": null,
          "author": {
            "id": 29027,
            "name": "Christine Gardner",
            "username": "gardnerchristine52",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/gardnerchristine52"
          },
          "created_at": "2020-01-07T20:25:27.101Z",
          "updated_at": "2020-01-07T20:25:27.101Z",
          "system": false,
          "noteable_id": 77513,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4425
        },
        "author": {
          "id": 29027,
          "name": "Christine Gardner",
          "username": "gardnerchristine52",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/gardnerchristine52"
        },
        "author_username": "gardnerchristine52",
        "user_id": 29027
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 680651,
        "target_iid": 680651,
        "target_type": "Note",
        "author_id": 29027,
        "target_title": "Saving to PDF changes text's font and size",
        "created_at": "2020-01-08T21:10:30.638Z",
        "note": {
          "id": 680651,
          "type": null,
          "body": "Did you flatten the layers?",
          "attachment": null,
          "author": {
            "id": 29027,
            "name": "Christine Gardner",
            "username": "gardnerchristine52",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/gardnerchristine52"
          },
          "created_at": "2020-01-08T21:10:29.892Z",
          "updated_at": "2020-01-08T21:10:29.892Z",
          "system": false,
          "noteable_id": 77513,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4425
        },
        "author": {
          "id": 29027,
          "name": "Christine Gardner",
          "username": "gardnerchristine52",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/gardnerchristine52"
        },
        "author_username": "gardnerchristine52",
        "user_id": 29027
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 680744,
        "target_iid": 680744,
        "target_type": "DiscussionNote",
        "author_id": 29027,
        "target_title": "Saving to PDF changes text's font and size",
        "created_at": "2020-01-08T22:45:18.777Z",
        "note": {
          "id": 680744,
          "type": "DiscussionNote",
          "body": "Thanks Jehan, I've found the tech talk here a bit hard to follow but I'll have a look at that. I don't understand what Michael's saying though. I have a book in print using the old version, with a font I couldn't convert to pdf on the new version, and it looks great.",
          "attachment": null,
          "author": {
            "id": 29027,
            "name": "Christine Gardner",
            "username": "gardnerchristine52",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/gardnerchristine52"
          },
          "created_at": "2020-01-08T22:45:18.107Z",
          "updated_at": "2020-01-08T22:45:18.107Z",
          "system": false,
          "noteable_id": 77513,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4425
        },
        "author": {
          "id": 29027,
          "name": "Christine Gardner",
          "username": "gardnerchristine52",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/64ad23329b808f3a1e9996517014b00f?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/gardnerchristine52"
        },
        "author_username": "gardnerchristine52",
        "user_id": 29027
      }
    ]
  },
  {
    "id": 29028,
    "events": [
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 674874,
        "target_iid": 674874,
        "target_type": "Note",
        "author_id": 29028,
        "target_title": "GIMP 2.10.14 does not start on W10 (affects users using gimp-2.10.14-setup-2.exe on nVidia GPUs)",
        "created_at": "2019-12-30T00:25:13.593Z",
        "note": {
          "id": 674874,
          "type": null,
          "body": "Here running less than a week windows 10 install and setup-1 works and setup-2 does not.  Radeon RX480 GPU, Ryzed 3960X CPU.  No AV installed beyond default MS one",
          "attachment": null,
          "author": {
            "id": 29028,
            "name": "vmpn",
            "username": "vmpn",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/d13a6571cf134aa7747fe47f6cd48908?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/vmpn"
          },
          "created_at": "2019-12-30T00:25:12.497Z",
          "updated_at": "2019-12-30T00:25:12.497Z",
          "system": false,
          "noteable_id": 77168,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4385
        },
        "author": {
          "id": 29028,
          "name": "vmpn",
          "username": "vmpn",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/d13a6571cf134aa7747fe47f6cd48908?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/vmpn"
        },
        "author_username": "vmpn",
        "user_id": 29028
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 675890,
        "target_iid": 675890,
        "target_type": "Note",
        "author_id": 29028,
        "target_title": "GIMP 2.10.14 does not start on W10 (affects users using gimp-2.10.14-setup-2.exe on nVidia GPUs)",
        "created_at": "2020-01-02T01:02:37.900Z",
        "note": {
          "id": 675890,
          "type": null,
          "body": "With ...setup-3.exe GIMP now works for me as well. Thank you",
          "attachment": null,
          "author": {
            "id": 29028,
            "name": "vmpn",
            "username": "vmpn",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/d13a6571cf134aa7747fe47f6cd48908?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/vmpn"
          },
          "created_at": "2020-01-02T01:02:36.702Z",
          "updated_at": "2020-01-02T01:02:36.702Z",
          "system": false,
          "noteable_id": 77168,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4385
        },
        "author": {
          "id": 29028,
          "name": "vmpn",
          "username": "vmpn",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/d13a6571cf134aa7747fe47f6cd48908?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/vmpn"
        },
        "author_username": "vmpn",
        "user_id": 29028
      }
    ]
  },
  {
    "id": 29029,
    "events": [
      {
        "project_id": 1894,
        "action_name": "opened",
        "target_id": 77511,
        "target_iid": 659,
        "target_type": "Issue",
        "author_id": 29029,
        "target_title": "IMAP/SMTP FQDN ONLY?",
        "created_at": "2019-12-30T01:50:17.470Z",
        "author": {
          "id": 29029,
          "name": "Sip Codec",
          "username": "sipcodec",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/c3b287adb5ffa1e43a080f39d76590d4?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/sipcodec"
        },
        "author_username": "sipcodec",
        "user_id": 29029
      },
      {
        "project_id": 1894,
        "action_name": "commented on",
        "target_id": 675263,
        "target_iid": 675263,
        "target_type": "Note",
        "author_id": 29029,
        "target_title": "IMAP/SMTP FQDN ONLY?",
        "created_at": "2019-12-30T20:05:47.649Z",
        "note": {
          "id": 675263,
          "type": null,
          "body": "Per screenshot FQDN is not accepted as well.![Selection_003](/uploads/1af8273e3877ec9aab1279078d82813e/Selection_003.png)",
          "attachment": null,
          "author": {
            "id": 29029,
            "name": "Sip Codec",
            "username": "sipcodec",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/c3b287adb5ffa1e43a080f39d76590d4?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/sipcodec"
          },
          "created_at": "2019-12-30T20:05:46.980Z",
          "updated_at": "2019-12-30T20:05:46.980Z",
          "system": false,
          "noteable_id": 77511,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 659
        },
        "author": {
          "id": 29029,
          "name": "Sip Codec",
          "username": "sipcodec",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/c3b287adb5ffa1e43a080f39d76590d4?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/sipcodec"
        },
        "author_username": "sipcodec",
        "user_id": 29029
      },
      {
        "project_id": 1894,
        "action_name": "commented on",
        "target_id": 675269,
        "target_iid": 675269,
        "target_type": "Note",
        "author_id": 29029,
        "target_title": "IMAP/SMTP FQDN ONLY?",
        "created_at": "2019-12-30T20:06:54.250Z",
        "note": {
          "id": 675269,
          "type": null,
          "body": "Would need fields to enter IMAP/SMTP ports as well since 127.0.0.1:1224 syntax is not accepted.",
          "attachment": null,
          "author": {
            "id": 29029,
            "name": "Sip Codec",
            "username": "sipcodec",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/c3b287adb5ffa1e43a080f39d76590d4?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/sipcodec"
          },
          "created_at": "2019-12-30T20:06:53.640Z",
          "updated_at": "2019-12-30T20:06:53.640Z",
          "system": false,
          "noteable_id": 77511,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 659
        },
        "author": {
          "id": 29029,
          "name": "Sip Codec",
          "username": "sipcodec",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/c3b287adb5ffa1e43a080f39d76590d4?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/sipcodec"
        },
        "author_username": "sipcodec",
        "user_id": 29029
      },
      {
        "project_id": 1894,
        "action_name": "commented on",
        "target_id": 675331,
        "target_iid": 675331,
        "target_type": "Note",
        "author_id": 29029,
        "target_title": "IMAP/SMTP FQDN ONLY?",
        "created_at": "2019-12-30T23:09:12.295Z",
        "note": {
          "id": 675331,
          "type": null,
          "body": "localhost:1224 is an illustration since I do not see additional fields to enter ports - Do not read too much into this. Simply put, when i type just \"localhost\", as you suggested (FQDN), the above error appears = \"Account not created...\"\n\nNot entirely sure what the error could be attributed to as every field is correct.",
          "attachment": null,
          "author": {
            "id": 29029,
            "name": "Sip Codec",
            "username": "sipcodec",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/c3b287adb5ffa1e43a080f39d76590d4?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/sipcodec"
          },
          "created_at": "2019-12-30T23:09:11.473Z",
          "updated_at": "2019-12-30T23:09:11.473Z",
          "system": false,
          "noteable_id": 77511,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 659
        },
        "author": {
          "id": 29029,
          "name": "Sip Codec",
          "username": "sipcodec",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/c3b287adb5ffa1e43a080f39d76590d4?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/sipcodec"
        },
        "author_username": "sipcodec",
        "user_id": 29029
      },
      {
        "project_id": 1894,
        "action_name": "commented on",
        "target_id": 675342,
        "target_iid": 675342,
        "target_type": "Note",
        "author_id": 29029,
        "target_title": "IMAP/SMTP FQDN ONLY?",
        "created_at": "2019-12-30T23:31:40.893Z",
        "note": {
          "id": 675342,
          "type": null,
          "body": "[Geary_Inspector_-_2019-12-30_17_20_28.txt](/uploads/5b070a872093e5d427dd8a93093a4445/Geary_Inspector_-_2019-12-30_17_20_28.txt)\n\n[Geary_Inspector_-_2019-12-30_17_20_14.txt](/uploads/6411d237959e8ef99aac6034b78a4e0e/Geary_Inspector_-_2019-12-30_17_20_14.txt)",
          "attachment": null,
          "author": {
            "id": 29029,
            "name": "Sip Codec",
            "username": "sipcodec",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/c3b287adb5ffa1e43a080f39d76590d4?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/sipcodec"
          },
          "created_at": "2019-12-30T23:31:40.282Z",
          "updated_at": "2019-12-30T23:31:40.282Z",
          "system": false,
          "noteable_id": 77511,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 659
        },
        "author": {
          "id": 29029,
          "name": "Sip Codec",
          "username": "sipcodec",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/c3b287adb5ffa1e43a080f39d76590d4?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/sipcodec"
        },
        "author_username": "sipcodec",
        "user_id": 29029
      },
      {
        "project_id": 1894,
        "action_name": "commented on",
        "target_id": 675343,
        "target_iid": 675343,
        "target_type": "Note",
        "author_id": 29029,
        "target_title": "IMAP/SMTP FQDN ONLY?",
        "created_at": "2019-12-30T23:35:28.725Z",
        "note": {
          "id": 675343,
          "type": null,
          "body": "[Geary_Inspector_-_2019-12-30_17_34_38.txt](/uploads/173e50a9e3a15dfc0b5a682dbc31dbc3/Geary_Inspector_-_2019-12-30_17_34_38.txt)\n\nReproduced a couple of times.",
          "attachment": null,
          "author": {
            "id": 29029,
            "name": "Sip Codec",
            "username": "sipcodec",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/c3b287adb5ffa1e43a080f39d76590d4?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/sipcodec"
          },
          "created_at": "2019-12-30T23:35:28.079Z",
          "updated_at": "2019-12-30T23:35:28.079Z",
          "system": false,
          "noteable_id": 77511,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 659
        },
        "author": {
          "id": 29029,
          "name": "Sip Codec",
          "username": "sipcodec",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/c3b287adb5ffa1e43a080f39d76590d4?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/sipcodec"
        },
        "author_username": "sipcodec",
        "user_id": 29029
      }
    ]
  },
  {
    "id": 29032,
    "events": []
  },
  {
    "id": 29033,
    "events": [
      {
        "project_id": 1857,
        "action_name": "opened",
        "target_id": 77514,
        "target_iid": 1737,
        "target_type": "Issue",
        "author_id": 29033,
        "target_title": "Build failure in DAAP plugin due to deprecated G_CONST_RETURN usage in libdmapsharing library",
        "created_at": "2019-12-30T05:55:26.043Z",
        "author": {
          "id": 29033,
          "name": "crvi",
          "username": "crvi",
          "state": "active",
          "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29033/avatar.png",
          "web_url": "https://gitlab.gnome.org/crvi"
        },
        "author_username": "crvi",
        "user_id": 29033
      },
      {
        "project_id": 1857,
        "action_name": "commented on",
        "target_id": 674984,
        "target_iid": 674984,
        "target_type": "Note",
        "author_id": 29033,
        "target_title": "Build failure in DAAP plugin due to deprecated G_CONST_RETURN usage in libdmapsharing library",
        "created_at": "2019-12-30T12:08:20.476Z",
        "note": {
          "id": 674984,
          "type": null,
          "body": "Hi .. the actual error message is a const error, which is in the bug description. Should this be included in the subject ?",
          "attachment": null,
          "author": {
            "id": 29033,
            "name": "crvi",
            "username": "crvi",
            "state": "active",
            "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29033/avatar.png",
            "web_url": "https://gitlab.gnome.org/crvi"
          },
          "created_at": "2019-12-30T12:08:19.918Z",
          "updated_at": "2019-12-30T12:17:19.692Z",
          "system": false,
          "noteable_id": 77514,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 1737
        },
        "author": {
          "id": 29033,
          "name": "crvi",
          "username": "crvi",
          "state": "active",
          "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29033/avatar.png",
          "web_url": "https://gitlab.gnome.org/crvi"
        },
        "author_username": "crvi",
        "user_id": 29033
      },
      {
        "project_id": 1857,
        "action_name": "commented on",
        "target_id": 675368,
        "target_iid": 675368,
        "target_type": "Note",
        "author_id": 29033,
        "target_title": "Build failure in DAAP plugin due to deprecated G_CONST_RETURN usage in libdmapsharing library",
        "created_at": "2019-12-31T05:52:49.994Z",
        "note": {
          "id": 675368,
          "type": null,
          "body": "The issue is due to recent glib versions ( glib 2.62 in Ubuntu 19.10 ) which have made G_CONST_RETURN a deprecated warning. With -Werror the compilation fails. Later versions of glib have a better warning message than just a \"error: const\" message. \n\nThe deprecation issue is fixed in libdmapsharing recently.\n\nhttps://gitlab.gnome.org/GNOME/libdmapsharing/commit/3d79874066e1a1c1bba646135cf5fc5811b0eb55\n\nA new release need to be pushed to fix the compiler error / warning.",
          "attachment": null,
          "author": {
            "id": 29033,
            "name": "crvi",
            "username": "crvi",
            "state": "active",
            "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29033/avatar.png",
            "web_url": "https://gitlab.gnome.org/crvi"
          },
          "created_at": "2019-12-31T05:52:49.294Z",
          "updated_at": "2019-12-31T05:55:23.915Z",
          "system": false,
          "noteable_id": 77514,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 1737
        },
        "author": {
          "id": 29033,
          "name": "crvi",
          "username": "crvi",
          "state": "active",
          "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29033/avatar.png",
          "web_url": "https://gitlab.gnome.org/crvi"
        },
        "author_username": "crvi",
        "user_id": 29033
      },
      {
        "project_id": 1857,
        "action_name": "opened",
        "target_id": 77600,
        "target_iid": 1738,
        "target_type": "Issue",
        "author_id": 29033,
        "target_title": "Android: Cannot remove tracks in Samsung Galaxy S10 after successfully adding tracks",
        "created_at": "2019-12-31T17:58:21.234Z",
        "author": {
          "id": 29033,
          "name": "crvi",
          "username": "crvi",
          "state": "active",
          "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29033/avatar.png",
          "web_url": "https://gitlab.gnome.org/crvi"
        },
        "author_username": "crvi",
        "user_id": 29033
      },
      {
        "project_id": 1857,
        "action_name": "commented on",
        "target_id": 675492,
        "target_iid": 675492,
        "target_type": "Note",
        "author_id": 29033,
        "target_title": "Android: Cannot remove tracks in Samsung Galaxy S10 after successfully adding tracks",
        "created_at": "2019-12-31T18:00:08.720Z",
        "note": {
          "id": 675492,
          "type": null,
          "body": "Rhythmbox debug log.\n\n[rhythmbox.log](/uploads/37229c5c2198e31143d742f185f20358/rhythmbox.log)",
          "attachment": null,
          "author": {
            "id": 29033,
            "name": "crvi",
            "username": "crvi",
            "state": "active",
            "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29033/avatar.png",
            "web_url": "https://gitlab.gnome.org/crvi"
          },
          "created_at": "2019-12-31T18:00:08.164Z",
          "updated_at": "2019-12-31T18:00:08.164Z",
          "system": false,
          "noteable_id": 77600,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 1738
        },
        "author": {
          "id": 29033,
          "name": "crvi",
          "username": "crvi",
          "state": "active",
          "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29033/avatar.png",
          "web_url": "https://gitlab.gnome.org/crvi"
        },
        "author_username": "crvi",
        "user_id": 29033
      },
      {
        "project_id": 1513,
        "action_name": "opened",
        "target_id": 77601,
        "target_iid": 9,
        "target_type": "Issue",
        "author_id": 29033,
        "target_title": "Make new release of libdmapsharing",
        "created_at": "2019-12-31T18:07:32.485Z",
        "author": {
          "id": 29033,
          "name": "crvi",
          "username": "crvi",
          "state": "active",
          "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29033/avatar.png",
          "web_url": "https://gitlab.gnome.org/crvi"
        },
        "author_username": "crvi",
        "user_id": 29033
      },
      {
        "project_id": 9713,
        "action_name": "created",
        "target_id": null,
        "target_iid": null,
        "target_type": null,
        "author_id": 29033,
        "target_title": null,
        "created_at": "2020-01-01T08:45:31.647Z",
        "author": {
          "id": 29033,
          "name": "crvi",
          "username": "crvi",
          "state": "active",
          "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29033/avatar.png",
          "web_url": "https://gitlab.gnome.org/crvi"
        },
        "author_username": "crvi",
        "user_id": 29033
      },
      {
        "project_id": 1513,
        "action_name": "commented on",
        "target_id": 675718,
        "target_iid": 675718,
        "target_type": "Note",
        "author_id": 29033,
        "target_title": "3.9.7: build fails",
        "created_at": "2020-01-01T17:10:43.141Z",
        "note": {
          "id": 675718,
          "type": null,
          "body": "```\n[...]$ ./configure --with-mdns=avahi --enable-gtk-doc --disable-tests\n[...]$ make\n...\n...\nMaking all in doc\nmake[2]: Entering directory '/home/sid/libdmapsharing/doc'\n  DOC   Preparing build\n  DOC   Scanning header files\n  DOC   Introspecting gobjects\n/usr/bin/ld: .libs/libdmapsharing-4.0-scan.o: in function `main':\nlibdmapsharing-4.0-scan.c:(.text.startup+0x199): undefined reference to `test_dmap_av_record_factory_get_type'\n/usr/bin/ld: libdmapsharing-4.0-scan.c:(.text.startup+0x1a5): undefined reference to `test_dmap_av_record_get_type'\n/usr/bin/ld: libdmapsharing-4.0-scan.c:(.text.startup+0x1b1): undefined reference to `test_dmap_container_db_get_type'\n/usr/bin/ld: libdmapsharing-4.0-scan.c:(.text.startup+0x1bd): undefined reference to `test_dmap_container_record_get_type'\n/usr/bin/ld: libdmapsharing-4.0-scan.c:(.text.startup+0x1c9): undefined reference to `test_dmap_db_get_type'\n/usr/bin/ld: libdmapsharing-4.0-scan.c:(.text.startup+0x1d5): undefined reference to `test_dmap_image_record_factory_get_type'\n/usr/bin/ld: libdmapsharing-4.0-scan.c:(.text.startup+0x1e1): undefined reference to `test_dmap_image_record_get_type'\ncollect2: error: ld returned 1 exit status\n2020-01-01 22:27:39,080:scangobj.py:execute_command:1200:WARNING:Linking scanner failed: 1, command: /bin/bash ../libtool --tag=CC --mode=link gcc -lgobject-2.0 -lglib-2.0 -O2 -Wall libdmapsharing-4.0-scan.lo ../libdmapsharing/libdmapsharing-4.0.la -lglib-2.0 -lgthread-2.0 -pthread -lglib-2.0 -lgobject-2.0 -lglib-2.0 -lavahi-common -lavahi-client -lavahi-glib -lglib-2.0 -lgstapp-1.0 -lgstbase-1.0 -lgstreamer-1.0 -lgobject-2.0 -lglib-2.0 -lsoup-2.4 -lgio-2.0 -lgobject-2.0 -lglib-2.0 -o libdmapsharing-4.0-scan\nmake[2]: *** [Makefile:758: scan-build.stamp] Error 1\nmake[2]: Leaving directory '/home/sid/libdmapsharing/doc'\nmake[1]: *** [Makefile:524: all-recursive] Error 1\nmake[1]: Leaving directory '/home/sid/libdmapsharing'\nmake: *** [Makefile:431: all] Error 2\n\n\n```",
          "attachment": null,
          "author": {
            "id": 29033,
            "name": "crvi",
            "username": "crvi",
            "state": "active",
            "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29033/avatar.png",
            "web_url": "https://gitlab.gnome.org/crvi"
          },
          "created_at": "2020-01-01T17:10:42.445Z",
          "updated_at": "2020-01-01T17:10:42.445Z",
          "system": false,
          "noteable_id": 69215,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 8
        },
        "author": {
          "id": 29033,
          "name": "crvi",
          "username": "crvi",
          "state": "active",
          "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29033/avatar.png",
          "web_url": "https://gitlab.gnome.org/crvi"
        },
        "author_username": "crvi",
        "user_id": 29033
      },
      {
        "project_id": 1513,
        "action_name": "commented on",
        "target_id": 675719,
        "target_iid": 675719,
        "target_type": "Note",
        "author_id": 29033,
        "target_title": "3.9.7: build fails",
        "created_at": "2020-01-01T17:12:33.694Z",
        "note": {
          "id": 675719,
          "type": null,
          "body": "The above error was on latest git.\n\n```\nsid@unstable:~/libdmapsharing$ git log | head\ncommit 9ae04e25cf792f2f7b4d3bd537bc778d7b05eabf\nAuthor: W. Michael Petullo <mike@flyn.org>\nDate:   Mon Dec 30 21:36:11 2019 -0500\n\n    Make more functions static\n```",
          "attachment": null,
          "author": {
            "id": 29033,
            "name": "crvi",
            "username": "crvi",
            "state": "active",
            "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29033/avatar.png",
            "web_url": "https://gitlab.gnome.org/crvi"
          },
          "created_at": "2020-01-01T17:12:33.063Z",
          "updated_at": "2020-01-01T17:12:53.729Z",
          "system": false,
          "noteable_id": 69215,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 8
        },
        "author": {
          "id": 29033,
          "name": "crvi",
          "username": "crvi",
          "state": "active",
          "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29033/avatar.png",
          "web_url": "https://gitlab.gnome.org/crvi"
        },
        "author_username": "crvi",
        "user_id": 29033
      },
      {
        "project_id": 1513,
        "action_name": "opened",
        "target_id": 77635,
        "target_iid": 10,
        "target_type": "Issue",
        "author_id": 29033,
        "target_title": "test-dmap-client.c: undefined reference to `test_dmap_db_new'",
        "created_at": "2020-01-01T17:34:59.248Z",
        "author": {
          "id": 29033,
          "name": "crvi",
          "username": "crvi",
          "state": "active",
          "avatar_url": "https://gitlab.gnome.org/uploads/-/system/user/avatar/29033/avatar.png",
          "web_url": "https://gitlab.gnome.org/crvi"
        },
        "author_username": "crvi",
        "user_id": 29033
      }
    ]
  },
  {
    "id": 29037,
    "events": []
  },
  {
    "id": 29038,
    "events": [
      {
        "project_id": 9708,
        "action_name": "created",
        "target_id": null,
        "target_iid": null,
        "target_type": null,
        "author_id": 29038,
        "target_title": null,
        "created_at": "2019-12-30T09:05:23.693Z",
        "author": {
          "id": 29038,
          "name": "Simone Mora",
          "username": "simomora84",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/33c5e71fd5ab9e575517cfd5b591a2a8?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/simomora84"
        },
        "author_username": "simomora84",
        "user_id": 29038
      }
    ]
  },
  {
    "id": 29040,
    "events": [
      {
        "project_id": 1898,
        "action_name": "opened",
        "target_id": 77519,
        "target_iid": 199,
        "target_type": "Issue",
        "author_id": 29040,
        "target_title": "Support cr3 (raw format by canon)",
        "created_at": "2019-12-30T11:06:34.400Z",
        "author": {
          "id": 29040,
          "name": "fabsmusicjunkie",
          "username": "fabsmusicjunkie",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/184c7cd59df4800206631f35f9c8075b?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/fabsmusicjunkie"
        },
        "author_username": "fabsmusicjunkie",
        "user_id": 29040
      }
    ]
  },
  {
    "id": 29042,
    "events": []
  },
  {
    "id": 29043,
    "events": [
      {
        "project_id": 274,
        "action_name": "opened",
        "target_id": 77521,
        "target_iid": 2157,
        "target_type": "Issue",
        "author_id": 29043,
        "target_title": "App crashing when clicked on Preferences",
        "created_at": "2019-12-30T11:58:09.805Z",
        "author": {
          "id": 29043,
          "name": "archDynasty",
          "username": "indahud",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/8163313fd5cac04beb554fcef641083d?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/indahud"
        },
        "author_username": "indahud",
        "user_id": 29043
      },
      {
        "project_id": 274,
        "action_name": "commented on",
        "target_id": 675254,
        "target_iid": 675254,
        "target_type": "Note",
        "author_id": 29043,
        "target_title": "App crashing when clicked on Preferences",
        "created_at": "2019-12-30T19:22:25.512Z",
        "note": {
          "id": 675254,
          "type": null,
          "body": "```\n[INFO] 2019-12-30 20:17:49 LastFM::__init__(): [Errno 2] No such file or directory: '/home/york/.local/share/lollypop/lastfm_queue.bin'\n[INFO] 2019-12-30 20:17:49 LastFM::__init__(): [Errno 2] No such file or directory: '/home/york/.local/share/lollypop/librefm_queue.bin'\n[WARNING] 2019-12-30 20:17:49 TodayBannerWidget::__get_today_album(): [Errno 2] No such file or directory: '/home/york/.local/share/lollypop/today.bin'\n[ERROR] 2019-12-30 20:17:49 TodayBannerWidget::__get_today_album(): no such column: storage_type\n[ERROR] 2019-12-30 20:17:49 TaskHelper::__run(): no such column: albums.storage_type: <function AlbumsPopularsBoxView.populate.<locals>.load at 0x7f2bcd8f5a60> -> {'callback': (<function AlbumsPopularsBoxView.populate.<locals>.on_load at 0x7f2bcd8f59d0>,)}\n[ERROR] 2019-12-30 20:17:49 TaskHelper::__run(): no such column: albums.storage_type: <function RoundedArtistsRandomView.populate.<locals>.load at 0x7f2bcd8f5ca0> -> {'callback': (<function RoundedArtistsRandomView.populate.<locals>.on_load at 0x7f2bcd8f5c10>,)}\n[ERROR] 2019-12-30 20:17:49 TaskHelper::__run(): no such column: storage_type: <function AlbumsRandomGenreBoxView.populate.<locals>.load at 0x7f2bcd8ff040> -> {'callback': (<function AlbumsRandomGenreBoxView.populate.<locals>.on_load at 0x7f2bcd8f5f70>,)}\n[ERROR] 2019-12-30 20:17:49 LastFM::__on_get_password(): no attributes\n[ERROR] 2019-12-30 20:17:49 LastFM::__on_get_password(): no attributes\n[INFO] 2019-12-30 20:17:50 Scan started\n[ERROR] 2019-12-30 20:17:50 TaskHelper::__run(): no such column: storage_type: <bound method CollectionScanner.__scan of <collectionscanner.CollectionScanner object at 0x7f2bcd8d9f40 (lollypop+collectionscanner+CollectionScanner at 0x55d1a6dc22c0)>> -> {}\nTraceback (most recent call last):\n  File \"/usr/lib/python3.8/site-packages/lollypop/application_actions.py\", line 200, in __on_settings_activate\n    dialog = SettingsDialog(param.get_int32())\n  File \"/usr/lib/python3.8/site-packages/lollypop/dialog_settings.py\", line 49, in __init__\n    collections_widget = CollectionsSettingsWidget()\n  File \"/usr/lib/python3.8/site-packages/lollypop/widgets_settings_collections.py\", line 125, in __init__\n    artists = App().artists.count()\n  File \"/usr/lib/python3.8/site-packages/lollypop/database_artists.py\", line 340, in count\n    result = sql.execute(\"SELECT COUNT(DISTINCT artists.rowid)\\\nsqlite3.OperationalError: no such column: albums.storage_type\n\n```",
          "attachment": null,
          "author": {
            "id": 29043,
            "name": "archDynasty",
            "username": "indahud",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/8163313fd5cac04beb554fcef641083d?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/indahud"
          },
          "created_at": "2019-12-30T19:22:25.226Z",
          "updated_at": "2019-12-30T19:22:25.226Z",
          "system": false,
          "noteable_id": 77521,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 2157
        },
        "author": {
          "id": 29043,
          "name": "archDynasty",
          "username": "indahud",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/8163313fd5cac04beb554fcef641083d?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/indahud"
        },
        "author_username": "indahud",
        "user_id": 29043
      }
    ]
  },
  {
    "id": 29047,
    "events": [
      {
        "project_id": 1848,
        "action_name": "opened",
        "target_id": 77524,
        "target_iid": 4426,
        "target_type": "Issue",
        "author_id": 29047,
        "target_title": "Add Backface Culling option to Map Object",
        "created_at": "2019-12-30T14:34:31.809Z",
        "author": {
          "id": 29047,
          "name": "SorteKanin",
          "username": "SorteKanin",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/cfbccca34073a722d789cd24084e4db4?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/SorteKanin"
        },
        "author_username": "SorteKanin",
        "user_id": 29047
      }
    ]
  },
  {
    "id": 29048,
    "events": []
  },
  {
    "id": 29049,
    "events": [
      {
        "project_id": 558,
        "action_name": "commented on",
        "target_id": 675096,
        "target_iid": 675096,
        "target_type": "Note",
        "author_id": 29049,
        "target_title": "Something went wrong appears when launched",
        "created_at": "2019-12-30T14:36:34.813Z",
        "note": {
          "id": 675096,
          "type": null,
          "body": "I got the issue since upgrading from Debian Stretch to Debian Buster\n\nI include the verboseoutput.txt\n\n[verboseoutput.txt](/uploads/27d4b4d2571a26b8cf7c5d8520c3421a/verboseoutput.txt)\n\n[EDIT]\nI removed gnome-software completely from the system (Debian Buster amd64) and re\u00efnstalled from the repository and now it works fine. So obviously something went wrong during the upgrade from Stretch to Buster (10.2).\n",
          "attachment": null,
          "author": {
            "id": 29049,
            "name": "Serge De Maseneer",
            "username": "SergeDeMaseneer",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/6b68bf237be4831bdced5f6d1b966e4c?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/SergeDeMaseneer"
          },
          "created_at": "2019-12-30T14:36:33.846Z",
          "updated_at": "2019-12-30T15:43:13.148Z",
          "system": false,
          "noteable_id": 9320,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 271
        },
        "author": {
          "id": 29049,
          "name": "Serge De Maseneer",
          "username": "SergeDeMaseneer",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/6b68bf237be4831bdced5f6d1b966e4c?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/SergeDeMaseneer"
        },
        "author_username": "SergeDeMaseneer",
        "user_id": 29049
      }
    ]
  },
  {
    "id": 29050,
    "events": [
      {
        "project_id": 658,
        "action_name": "opened",
        "target_id": 77526,
        "target_iid": 1971,
        "target_type": "Issue",
        "author_id": 29050,
        "target_title": "Glib Installation Error:-  meson.build:1919:2: ERROR: Dependency \"mount\" not found, tried pkgconfig and cmake",
        "created_at": "2019-12-30T14:45:27.952Z",
        "author": {
          "id": 29050,
          "name": "bhimbbiswa",
          "username": "bhimbbiswa",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/f4d14c94c664345a1e1bbabc2f1782a4?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/bhimbbiswa"
        },
        "author_username": "bhimbbiswa",
        "user_id": 29050
      }
    ]
  },
  {
    "id": 29052,
    "events": []
  },
  {
    "id": 29053,
    "events": [
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 675131,
        "target_iid": 675131,
        "target_type": "Note",
        "author_id": 29053,
        "target_title": "Incorrect Histogram Arithmetic Mean Values",
        "created_at": "2019-12-30T15:59:39.515Z",
        "note": {
          "id": 675131,
          "type": null,
          "body": "How many people are aware of equation **mean_absolute = mean \\ (2^bitsPerChannel - 1)**?\n\nWhy not provide two fields for the mean values of histograms, and let users decide which units they want to use?\n\nE.g.:\nOne field for standard RGB units [0 to 255].\nOne field for RGB units stated as [0 to 1.0].\n\nHere is an example: http://colorizer.org\n\nThank you for your consideration,\n\nDaniel Guibord",
          "attachment": null,
          "author": {
            "id": 29053,
            "name": "Daniel Guibord",
            "username": "danielguibord",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/fc5e29aa5a655fe09b475ae22172bee5?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/danielguibord"
          },
          "created_at": "2019-12-30T15:59:38.680Z",
          "updated_at": "2019-12-30T21:21:46.556Z",
          "system": false,
          "noteable_id": 76530,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4332
        },
        "author": {
          "id": 29053,
          "name": "Daniel Guibord",
          "username": "danielguibord",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/fc5e29aa5a655fe09b475ae22172bee5?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/danielguibord"
        },
        "author_username": "danielguibord",
        "user_id": 29053
      }
    ]
  },
  {
    "id": 29058,
    "events": [
      {
        "project_id": 660,
        "action_name": "commented on",
        "target_id": 675298,
        "target_iid": 675298,
        "target_type": "Note",
        "author_id": 29058,
        "target_title": "Unnecessary Delays Recovering from Screen Lock",
        "created_at": "2019-12-30T21:22:18.441Z",
        "note": {
          "id": 675298,
          "type": null,
          "body": "I hate to complain, but I'm so tired of this delay. Is there a work-around for this?",
          "attachment": null,
          "author": {
            "id": 29058,
            "name": "Lonnie Best",
            "username": "Lonniebiz2",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/8599b881b4fd54e0a81578dead6faf90?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/Lonniebiz2"
          },
          "created_at": "2019-12-30T21:22:17.635Z",
          "updated_at": "2019-12-30T21:22:17.635Z",
          "system": false,
          "noteable_id": 75004,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 757
        },
        "author": {
          "id": 29058,
          "name": "Lonnie Best",
          "username": "Lonniebiz2",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/8599b881b4fd54e0a81578dead6faf90?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/Lonniebiz2"
        },
        "author_username": "Lonniebiz2",
        "user_id": 29058
      }
    ]
  },
  {
    "id": 29059,
    "events": [
      {
        "project_id": 544,
        "action_name": "opened",
        "target_id": 77570,
        "target_iid": 269,
        "target_type": "Issue",
        "author_id": 29059,
        "target_title": "Trash Icon and Desktop Icons",
        "created_at": "2019-12-30T22:02:24.308Z",
        "author": {
          "id": 29059,
          "name": "Celtic Fox",
          "username": "ecs.tech19",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/c9ca11992975481bf7160bf1c1676d20?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/ecs.tech19"
        },
        "author_username": "ecs.tech19",
        "user_id": 29059
      }
    ]
  },
  {
    "id": 29060,
    "events": [
      {
        "project_id": 1596,
        "action_name": "opened",
        "target_id": 77572,
        "target_iid": 100,
        "target_type": "Issue",
        "author_id": 29060,
        "target_title": "JPEG rotation of lengths not divisible by 8 is incorrect",
        "created_at": "2019-12-30T22:14:06.681Z",
        "author": {
          "id": 29060,
          "name": "Peter J\u00fano\u0161",
          "username": "petoju",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/5e7ea1aa6ea9c1d68754d3b487b82f68?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/petoju"
        },
        "author_username": "petoju",
        "user_id": 29060
      }
    ]
  },
  {
    "id": 29062,
    "events": [
      {
        "project_id": 1848,
        "action_name": "opened",
        "target_id": 77573,
        "target_iid": 4428,
        "target_type": "Issue",
        "author_id": 29062,
        "target_title": "Plug-in crashed: \"file-png.exe\" (C:\\Program Files\\GIMP 2\\lib\\gimp\\2.0\\plug-ins\\file-png\\file-png.exe)",
        "created_at": "2019-12-30T22:35:40.907Z",
        "author": {
          "id": 29062,
          "name": "RichieMuse",
          "username": "rvyse2017",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/ef2dbc0ce311d9f06d3e50c823795a9a?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/rvyse2017"
        },
        "author_username": "rvyse2017",
        "user_id": 29062
      },
      {
        "project_id": 1848,
        "action_name": "closed",
        "target_id": 77573,
        "target_iid": 4428,
        "target_type": "Issue",
        "author_id": 29062,
        "target_title": "Plug-in crashed: \"file-png.exe\" (C:\\Program Files\\GIMP 2\\lib\\gimp\\2.0\\plug-ins\\file-png\\file-png.exe)",
        "created_at": "2019-12-30T22:43:20.017Z",
        "author": {
          "id": 29062,
          "name": "RichieMuse",
          "username": "rvyse2017",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/ef2dbc0ce311d9f06d3e50c823795a9a?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/rvyse2017"
        },
        "author_username": "rvyse2017",
        "user_id": 29062
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 675317,
        "target_iid": 675317,
        "target_type": "Note",
        "author_id": 29062,
        "target_title": "Plug-in crashed: \"file-png.exe\" (C:\\Program Files\\GIMP 2\\lib\\gimp\\2.0\\plug-ins\\file-png\\file-png.exe)",
        "created_at": "2019-12-30T22:46:06.391Z",
        "note": {
          "id": 675317,
          "type": null,
          "body": "OK so you need to **run the application as Admin**.\nThe application was still a bit glitchy as it doesn't always open but running as Admin brought up the export menu.",
          "attachment": null,
          "author": {
            "id": 29062,
            "name": "RichieMuse",
            "username": "rvyse2017",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/ef2dbc0ce311d9f06d3e50c823795a9a?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/rvyse2017"
          },
          "created_at": "2019-12-30T22:46:05.656Z",
          "updated_at": "2019-12-30T22:46:05.656Z",
          "system": false,
          "noteable_id": 77573,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4428
        },
        "author": {
          "id": 29062,
          "name": "RichieMuse",
          "username": "rvyse2017",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/ef2dbc0ce311d9f06d3e50c823795a9a?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/rvyse2017"
        },
        "author_username": "rvyse2017",
        "user_id": 29062
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 675322,
        "target_iid": 675322,
        "target_type": "Note",
        "author_id": 29062,
        "target_title": "Plug-in crashed: \"file-png.exe\" (C:\\Program Files\\GIMP 2\\lib\\gimp\\2.0\\plug-ins\\file-png\\file-png.exe)",
        "created_at": "2019-12-30T22:51:31.148Z",
        "note": {
          "id": 675322,
          "type": null,
          "body": "Ok so I have **reopened ** this because I opened my project and I successfully exported my image. Great.\nNow i made some changes and tried to export to a new file. I get the same error as before.\nThe application is open as ADMIN, the first export worked now it doesn't work.\nRe-Opening the application doesn't work.",
          "attachment": null,
          "author": {
            "id": 29062,
            "name": "RichieMuse",
            "username": "rvyse2017",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/ef2dbc0ce311d9f06d3e50c823795a9a?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/rvyse2017"
          },
          "created_at": "2019-12-30T22:51:30.552Z",
          "updated_at": "2019-12-30T22:51:59.880Z",
          "system": false,
          "noteable_id": 77573,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4428
        },
        "author": {
          "id": 29062,
          "name": "RichieMuse",
          "username": "rvyse2017",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/ef2dbc0ce311d9f06d3e50c823795a9a?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/rvyse2017"
        },
        "author_username": "rvyse2017",
        "user_id": 29062
      },
      {
        "project_id": 1848,
        "action_name": "opened",
        "target_id": 77573,
        "target_iid": 4428,
        "target_type": "Issue",
        "author_id": 29062,
        "target_title": "Plug-in crashed: \"file-png.exe\" (C:\\Program Files\\GIMP 2\\lib\\gimp\\2.0\\plug-ins\\file-png\\file-png.exe)",
        "created_at": "2019-12-30T22:51:32.966Z",
        "author": {
          "id": 29062,
          "name": "RichieMuse",
          "username": "rvyse2017",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/ef2dbc0ce311d9f06d3e50c823795a9a?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/rvyse2017"
        },
        "author_username": "rvyse2017",
        "user_id": 29062
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 675731,
        "target_iid": 675731,
        "target_type": "Note",
        "author_id": 29062,
        "target_title": "Plug-in crashed: \"file-png.exe\" (C:\\Program Files\\GIMP 2\\lib\\gimp\\2.0\\plug-ins\\file-png\\file-png.exe)",
        "created_at": "2020-01-01T17:34:24.035Z",
        "note": {
          "id": 675731,
          "type": null,
          "body": "So for example I have come back today, a couple of days later from when I created a little project.\nI simply follow these steps:\nFile > Export > (set the file path and file name example.png) then Export\nEverything works fine and the file has exported.\n\nHowever, now I change something in my design\nRepeat the exact same steps\nFile > Export > (set the file path and file name example2.png) then Export and I get the same error as reported in the original post.\n\n> Plug-in crashed: \"file-png.exe\"\n> (C:\\Program Files\\GIMP 2\\lib\\gimp\\2.0\\plug-ins\\file-png\\file-png.exe)\n> \n> The dying plug-in may have messed up GIMP's internal state. You may want to save your images and restart GIMP to be on the safe side.\n\nI have this application on my laptop and it's fine, it's also running windows 10, i'm not sure on the version as I installed it a while ago. Could this be a release issue?",
          "attachment": null,
          "author": {
            "id": 29062,
            "name": "RichieMuse",
            "username": "rvyse2017",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/ef2dbc0ce311d9f06d3e50c823795a9a?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/rvyse2017"
          },
          "created_at": "2020-01-01T17:34:23.206Z",
          "updated_at": "2020-01-01T17:34:23.206Z",
          "system": false,
          "noteable_id": 77573,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4428
        },
        "author": {
          "id": 29062,
          "name": "RichieMuse",
          "username": "rvyse2017",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/ef2dbc0ce311d9f06d3e50c823795a9a?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/rvyse2017"
        },
        "author_username": "rvyse2017",
        "user_id": 29062
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 675802,
        "target_iid": 675802,
        "target_type": "Note",
        "author_id": 29062,
        "target_title": "Plug-in crashed: \"file-png.exe\" (C:\\Program Files\\GIMP 2\\lib\\gimp\\2.0\\plug-ins\\file-png\\file-png.exe)",
        "created_at": "2020-01-01T21:16:20.120Z",
        "note": {
          "id": 675802,
          "type": null,
          "body": "I uninstalled the latest version again and reinstalled to no avail.\n**I have installed an older version of GIMP and the problem is not present.**\nI will use the older version until this is fixed.",
          "attachment": null,
          "author": {
            "id": 29062,
            "name": "RichieMuse",
            "username": "rvyse2017",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/ef2dbc0ce311d9f06d3e50c823795a9a?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/rvyse2017"
          },
          "created_at": "2020-01-01T21:16:19.336Z",
          "updated_at": "2020-01-01T21:16:53.575Z",
          "system": false,
          "noteable_id": 77573,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4428
        },
        "author": {
          "id": 29062,
          "name": "RichieMuse",
          "username": "rvyse2017",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/ef2dbc0ce311d9f06d3e50c823795a9a?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/rvyse2017"
        },
        "author_username": "rvyse2017",
        "user_id": 29062
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 676158,
        "target_iid": 676158,
        "target_type": "Note",
        "author_id": 29062,
        "target_title": "Plug-in crashed: \"file-png.exe\" (C:\\Program Files\\GIMP 2\\lib\\gimp\\2.0\\plug-ins\\file-png\\file-png.exe)",
        "created_at": "2020-01-02T19:14:22.190Z",
        "note": {
          "id": 676158,
          "type": null,
          "body": "I am using V2.8 NOW on the PC.\nI have GIMP **2.10.12** on my laptop which I was using today and it was fine on that.\nThe latest version is **2.10.14**, i'm sure it works because there must be loads of people using it but it's strange it won't work for me, but installing a different version works.\n\n*If this is a bug i'm more than happy to test solutions out or help pin point the problem but if i'm just unlucky and it's only me then I will use an older version for now.*\n\nThanks for your help.\n\n**UPDATE**\nI have installed 2.10.12 on the PC and it is working perfectly which gets me thinking is it something to do with 2.10.14?\nEvery other version had worked fine.",
          "attachment": null,
          "author": {
            "id": 29062,
            "name": "RichieMuse",
            "username": "rvyse2017",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/ef2dbc0ce311d9f06d3e50c823795a9a?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/rvyse2017"
          },
          "created_at": "2020-01-02T19:14:21.459Z",
          "updated_at": "2020-01-02T19:22:09.278Z",
          "system": false,
          "noteable_id": 77573,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4428
        },
        "author": {
          "id": 29062,
          "name": "RichieMuse",
          "username": "rvyse2017",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/ef2dbc0ce311d9f06d3e50c823795a9a?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/rvyse2017"
        },
        "author_username": "rvyse2017",
        "user_id": 29062
      }
    ]
  },
  {
    "id": 29063,
    "events": [
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 675350,
        "target_iid": 675350,
        "target_type": "Note",
        "author_id": 29063,
        "target_title": "Saving to PDF changes text's font and size",
        "created_at": "2019-12-31T01:31:34.507Z",
        "note": {
          "id": 675350,
          "type": null,
          "body": "I have an xcf file which I've been working on for a few years using older versions of GIMP. I opened it in version 2.10.14 today and tried making a few changes, but there are a lot of problems that I'm noticing, perhaps related to this.\n\n1) The font sizes within the file seems to be corrupted somehow. That is, a text block which was 90 px in the past, when being clicked on, is shown as 21 px. The preview is still at 90 px. But the moment I click the dragbars to resize the text layer, it immediately shrinks to 21px and then I have to change the fonts manually again.\n\n2) When exporting to PNG, everything is fine.\n\n3) When exporting to PDF, these problems are happening\n\n    * The font sizes are all small.\n    * Hidden layers are not omitted.\n\n\nI'm attaching the latest xcf file and the old and new pdf outputs.\n\n(Unfortunately I don't have the backup of the old xcf file)\n\n[english-v2.pdf](/uploads/16cdd6ba593bb44a5fbd5c18a706cab5/english-v2.pdf)\n\n[english.pdf](/uploads/3aa6bd1d9682d57f35be7d30535de061/english.pdf)\n\n[poster1-modified.xcf](/uploads/30f0520728b158be59f8353fbf5d9ed0/poster1-modified.xcf)",
          "attachment": null,
          "author": {
            "id": 29063,
            "name": "Akshay S Dinesh",
            "username": "asdofindia",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/0b038a1401aa8b721d837add33efe913?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/asdofindia"
          },
          "created_at": "2019-12-31T01:31:33.644Z",
          "updated_at": "2019-12-31T01:31:33.644Z",
          "system": false,
          "noteable_id": 77513,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4425
        },
        "author": {
          "id": 29063,
          "name": "Akshay S Dinesh",
          "username": "asdofindia",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/0b038a1401aa8b721d837add33efe913?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/asdofindia"
        },
        "author_username": "asdofindia",
        "user_id": 29063
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 675899,
        "target_iid": 675899,
        "target_type": "Note",
        "author_id": 29063,
        "target_title": "Saving to PDF changes text's font and size",
        "created_at": "2020-01-02T02:32:44.747Z",
        "note": {
          "id": 675899,
          "type": null,
          "body": "I didn't know it was different because the bug report sounded the same and was more or less on the same day and therefore I didn't want to create a duplicate.\n\nI included information about omit hidden layers bug because I thought that maybe contributory.\n\nShall I open another issue for the text size problem in my xcf?",
          "attachment": null,
          "author": {
            "id": 29063,
            "name": "Akshay S Dinesh",
            "username": "asdofindia",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/0b038a1401aa8b721d837add33efe913?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/asdofindia"
          },
          "created_at": "2020-01-02T02:32:43.968Z",
          "updated_at": "2020-01-02T02:32:43.968Z",
          "system": false,
          "noteable_id": 77513,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4425
        },
        "author": {
          "id": 29063,
          "name": "Akshay S Dinesh",
          "username": "asdofindia",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/0b038a1401aa8b721d837add33efe913?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/asdofindia"
        },
        "author_username": "asdofindia",
        "user_id": 29063
      }
    ]
  },
  {
    "id": 29064,
    "events": [
      {
        "project_id": 1593,
        "action_name": "commented on",
        "target_id": 675364,
        "target_iid": 675364,
        "target_type": "Note",
        "author_id": 29064,
        "target_title": "recurrent error about tracker-extract unable inserting multiple values",
        "created_at": "2019-12-31T04:34:32.594Z",
        "note": {
          "id": 675364,
          "type": null,
          "body": "I am getting the error:\n\n```\ntracker-extract: Could not insert metadata for item \"file:///home/andy/Documents/misc/second-floor-gimp.jpg\": Unable to insert multiple values for subject `urn:uuid:dfa5e52c-9541-0133-831a-fa7e24eebbd6' and single valued property `nmm:exposureTime' (old_value: '0.026316', new value: '0.034483')\n```\n\nI tried running:\n\n```\ntracker reset -e -f '/home/andy/Documents/misc/second-floor-gimp.jpg'\n```\nThat had no effect. There are over 40,000 entries in the last two days. I'm running Ubuntu 19.04.\n\n```\nlibtracker-control-2.0-0/disco,now 2.1.8-2 amd64 [installed,automatic]\nlibtracker-miner-2.0-0/disco,now 2.1.8-2 amd64 [installed,automatic]\nlibtracker-sparql-2.0-0/disco,now 2.1.8-2 amd64 [installed,automatic]\nlibxatracker2/disco-updates,now 19.0.8-0ubuntu0~19.04.1 amd64 [installed,automatic]\ntracker-extract/disco-updates,now 2.1.6-1ubuntu1~19.04 amd64 [installed,automatic]\ntracker-miner-fs/disco-updates,now 2.1.6-1ubuntu1~19.04 amd64 [installed,automatic]\ntracker/disco,now 2.1.8-2 amd64 [installed,automatic]\n```\nI saw some reference to .trackerignore. I put that in the directory. No change. It seems there was a GUI, but it does not exist in Unbuntu 19.04. The settings are very limited;all or nothing.\nFor now I have killed it.\n\nThanksm, Andy",
          "attachment": null,
          "author": {
            "id": 29064,
            "name": "Andy Howell",
            "username": "AndyHowell",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/7bd21a2de1b2df8d8579277debfad68d?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/AndyHowell"
          },
          "created_at": "2019-12-31T04:34:31.867Z",
          "updated_at": "2019-12-31T04:34:31.867Z",
          "system": false,
          "noteable_id": 76783,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 164
        },
        "author": {
          "id": 29064,
          "name": "Andy Howell",
          "username": "AndyHowell",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/7bd21a2de1b2df8d8579277debfad68d?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/AndyHowell"
        },
        "author_username": "AndyHowell",
        "user_id": 29064
      }
    ]
  },
  {
    "id": 29066,
    "events": [
      {
        "project_id": 1848,
        "action_name": "opened",
        "target_id": 77578,
        "target_iid": 4431,
        "target_type": "Issue",
        "author_id": 29066,
        "target_title": "Gimp Crush when trying to draw foreground using  foreground select tool",
        "created_at": "2019-12-31T06:39:01.281Z",
        "author": {
          "id": 29066,
          "name": "Niloy Barmon",
          "username": "niloybarmon.vu",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/bad2ccf5d933dd01518bfa8392522e79?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/niloybarmon.vu"
        },
        "author_username": "niloybarmon.vu",
        "user_id": 29066
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 676003,
        "target_iid": 676003,
        "target_type": "DiscussionNote",
        "author_id": 29066,
        "target_title": "Gimp Crush when trying to draw foreground using  foreground select tool",
        "created_at": "2020-01-02T11:39:35.383Z",
        "note": {
          "id": 676003,
          "type": "DiscussionNote",
          "body": "How to do that? \nI don't know :(",
          "attachment": null,
          "author": {
            "id": 29066,
            "name": "Niloy Barmon",
            "username": "niloybarmon.vu",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/bad2ccf5d933dd01518bfa8392522e79?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/niloybarmon.vu"
          },
          "created_at": "2020-01-02T11:39:34.471Z",
          "updated_at": "2020-01-02T11:39:34.471Z",
          "system": false,
          "noteable_id": 77578,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4431
        },
        "author": {
          "id": 29066,
          "name": "Niloy Barmon",
          "username": "niloybarmon.vu",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/bad2ccf5d933dd01518bfa8392522e79?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/niloybarmon.vu"
        },
        "author_username": "niloybarmon.vu",
        "user_id": 29066
      }
    ]
  },
  {
    "id": 29067,
    "events": [
      {
        "project_id": 1848,
        "action_name": "opened",
        "target_id": 77579,
        "target_iid": 4432,
        "target_type": "Issue",
        "author_id": 29067,
        "target_title": "Dot for dot won't disable",
        "created_at": "2019-12-31T07:15:17.446Z",
        "author": {
          "id": 29067,
          "name": "Brian van der Spuy",
          "username": "brianvds",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/207a28b89665e5faf4ada25f55f6028c?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/brianvds"
        },
        "author_username": "brianvds",
        "user_id": 29067
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 675612,
        "target_iid": 675612,
        "target_type": "Note",
        "author_id": 29067,
        "target_title": "Dot for dot won't disable",
        "created_at": "2020-01-01T03:41:47.953Z",
        "note": {
          "id": 675612,
          "type": null,
          "body": "Very strange: the bug reproduced perfectly well. Two restarts later, it spontaneously resolved. I'm kind of mystified, but it seems sorted now. Thanks for the input!",
          "attachment": null,
          "author": {
            "id": 29067,
            "name": "Brian van der Spuy",
            "username": "brianvds",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/207a28b89665e5faf4ada25f55f6028c?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/brianvds"
          },
          "created_at": "2020-01-01T03:41:47.429Z",
          "updated_at": "2020-01-01T03:41:47.429Z",
          "system": false,
          "noteable_id": 77579,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4432
        },
        "author": {
          "id": 29067,
          "name": "Brian van der Spuy",
          "username": "brianvds",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/207a28b89665e5faf4ada25f55f6028c?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/brianvds"
        },
        "author_username": "brianvds",
        "user_id": 29067
      }
    ]
  },
  {
    "id": 29069,
    "events": [
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 675377,
        "target_iid": 675377,
        "target_type": "Note",
        "author_id": 29069,
        "target_title": "GIMP 2.10.14 fails to start on Windows",
        "created_at": "2019-12-31T08:28:44.121Z",
        "note": {
          "id": 675377,
          "type": null,
          "body": "Very similar issue to RePlay.be\n\nGIMP simply will not launch. No output from gimp-2.10.exe\n\nHours old install of Windows 10 Pro. No antivirus software except for Windows Defender. I do not use Sophos.\n\nWindows 10 version 1909\n\nGimp version 2.10.14\n\n```\nC:\\Program Files\\GIMP 2\\bin>gimp-console-2.10.exe\nGEGL-Message: 03:02:39.854: Module 'C:\\Program Files\\GIMP 2\\lib\\gegl-0.4\\ff-load.dll' load error: 'C:\\Program Files\\GIMP 2\\lib\\gegl-0.4\\ff-load.dll': The specified module could not be found.\nGEGL-Message: 03:02:39.862: Module 'C:\\Program Files\\GIMP 2\\lib\\gegl-0.4\\ff-save.dll' load error: 'C:\\Program Files\\GIMP 2\\lib\\gegl-0.4\\ff-save.dll': The specified module could not be found.\nException code=0xc0000005 flags=0x0 at 0x00007FF9271472A6. Access violation - attempting to write data at address 0x0000000000000418\nException code=0xc0000005 flags=0x0 at 0x00007FF9271472A6. Access violation - attempting to write data at address 0x0000000000000418\nException code=0xc0000005 flags=0x0 at 0x00007FF9271472A6. Access violation - attempting to write data at address 0x0000000000000418\nException code=0xc0000005 flags=0x0 at 0x00007FF9271472A6\n```\n\nAs per ollielowson, I installed the heif image extension. This did not fix it for me.\n\n---\n\nGIMP 2.8.22 works.",
          "attachment": null,
          "author": {
            "id": 29069,
            "name": "Joe Haley",
            "username": "joehaley",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/012ce05b0a3590727b1e962e2d1e2bda?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/joehaley"
          },
          "created_at": "2019-12-31T08:28:43.070Z",
          "updated_at": "2019-12-31T08:33:19.882Z",
          "system": false,
          "noteable_id": 76289,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4304
        },
        "author": {
          "id": 29069,
          "name": "Joe Haley",
          "username": "joehaley",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/012ce05b0a3590727b1e962e2d1e2bda?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/joehaley"
        },
        "author_username": "joehaley",
        "user_id": 29069
      }
    ]
  },
  {
    "id": 29070,
    "events": [
      {
        "project_id": 1598,
        "action_name": "commented on",
        "target_id": 675381,
        "target_iid": 675381,
        "target_type": "Note",
        "author_id": 29070,
        "target_title": "file-roller looks at file extention not file header to determine type",
        "created_at": "2019-12-31T09:06:48.779Z",
        "note": {
          "id": 675381,
          "type": null,
          "body": "Another way to reproduce is to try to open a libre office PPTX (or DOCX, etc...) document.\n*  keep the pptx file extension and you get the \"Archive type not supported\" error message,\n*  rename to zip (since header 2 bytes are \"PK\") and file roller opens gracefully.",
          "attachment": null,
          "author": {
            "id": 29070,
            "name": "Alain Pannetier",
            "username": "alainpannetier",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/60fdc9673db87ee6d185f0327a6eb978?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/alainpannetier"
          },
          "created_at": "2019-12-31T09:06:48.056Z",
          "updated_at": "2019-12-31T09:06:48.056Z",
          "system": false,
          "noteable_id": 75808,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 57
        },
        "author": {
          "id": 29070,
          "name": "Alain Pannetier",
          "username": "alainpannetier",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/60fdc9673db87ee6d185f0327a6eb978?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/alainpannetier"
        },
        "author_username": "alainpannetier",
        "user_id": 29070
      }
    ]
  },
  {
    "id": 29071,
    "events": []
  },
  {
    "id": 29074,
    "events": []
  },
  {
    "id": 29076,
    "events": []
  },
  {
    "id": 29077,
    "events": [
      {
        "project_id": 1,
        "action_name": "opened",
        "target_id": 77593,
        "target_iid": 1327,
        "target_type": "Issue",
        "author_id": 29077,
        "target_title": "Wrong number of files to be/being copied when ignoring older files",
        "created_at": "2019-12-31T13:45:38.910Z",
        "author": {
          "id": 29077,
          "name": "Nelson Teixeira",
          "username": "nelson777",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/8d6e91fa8f76c8f19e8ed6f8a9f330c5?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/nelson777"
        },
        "author_username": "nelson777",
        "user_id": 29077
      }
    ]
  },
  {
    "id": 29079,
    "events": [
      {
        "project_id": 1,
        "action_name": "opened",
        "target_id": 77595,
        "target_iid": 1328,
        "target_type": "Issue",
        "author_id": 29079,
        "target_title": "While loading a directory, left button over folder breadcrumb to paste is blinking....and i can't paste",
        "created_at": "2019-12-31T15:02:34.552Z",
        "author": {
          "id": 29079,
          "name": "felisucoibi",
          "username": "felisucoibi",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/b8826fb04ce335da90dd4790f28790df?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/felisucoibi"
        },
        "author_username": "felisucoibi",
        "user_id": 29079
      },
      {
        "project_id": 1,
        "action_name": "opened",
        "target_id": 77596,
        "target_iid": 1329,
        "target_type": "Issue",
        "author_id": 29079,
        "target_title": "Allow search results for files to be ordered by size or file name",
        "created_at": "2019-12-31T15:03:42.923Z",
        "author": {
          "id": 29079,
          "name": "felisucoibi",
          "username": "felisucoibi",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/b8826fb04ce335da90dd4790f28790df?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/felisucoibi"
        },
        "author_username": "felisucoibi",
        "user_id": 29079
      },
      {
        "project_id": 1,
        "action_name": "opened",
        "target_id": 77597,
        "target_iid": 1330,
        "target_type": "Issue",
        "author_id": 29079,
        "target_title": "In search result left click and ADD an option to select all search results.",
        "created_at": "2019-12-31T15:06:08.457Z",
        "author": {
          "id": 29079,
          "name": "felisucoibi",
          "username": "felisucoibi",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/b8826fb04ce335da90dd4790f28790df?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/felisucoibi"
        },
        "author_username": "felisucoibi",
        "user_id": 29079
      },
      {
        "project_id": 1,
        "action_name": "commented on",
        "target_id": 675742,
        "target_iid": 675742,
        "target_type": "Note",
        "author_id": 29079,
        "target_title": "While loading a directory, left button over folder breadcrumb to paste is blinking....and i can't paste",
        "created_at": "2020-01-01T18:26:02.402Z",
        "note": {
          "id": 675742,
          "type": null,
          "body": "When a directory is loading and calculating total files inside other directories the copy paste blinks and is impossible to use the button.\nNautilus: 3.34.1\nUbuntu 19.10\n![image](/uploads/5ab85abdadc5e9e26c2f63e381658016/image.png)",
          "attachment": null,
          "author": {
            "id": 29079,
            "name": "felisucoibi",
            "username": "felisucoibi",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/b8826fb04ce335da90dd4790f28790df?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/felisucoibi"
          },
          "created_at": "2020-01-01T18:26:01.298Z",
          "updated_at": "2020-01-01T18:26:01.298Z",
          "system": false,
          "noteable_id": 77595,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 1328
        },
        "author": {
          "id": 29079,
          "name": "felisucoibi",
          "username": "felisucoibi",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/b8826fb04ce335da90dd4790f28790df?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/felisucoibi"
        },
        "author_username": "felisucoibi",
        "user_id": 29079
      },
      {
        "project_id": 1,
        "action_name": "commented on",
        "target_id": 675743,
        "target_iid": 675743,
        "target_type": "Note",
        "author_id": 29079,
        "target_title": "Allow search results for files to be ordered by size or file name",
        "created_at": "2020-01-01T18:27:31.177Z",
        "note": {
          "id": 675743,
          "type": null,
          "body": "Is not \"allow search\", is \"make it work\", because the sort column is there, but just don't work like it does when not searching.",
          "attachment": null,
          "author": {
            "id": 29079,
            "name": "felisucoibi",
            "username": "felisucoibi",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/b8826fb04ce335da90dd4790f28790df?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/felisucoibi"
          },
          "created_at": "2020-01-01T18:27:30.541Z",
          "updated_at": "2020-01-01T18:27:30.541Z",
          "system": false,
          "noteable_id": 77596,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 1329
        },
        "author": {
          "id": 29079,
          "name": "felisucoibi",
          "username": "felisucoibi",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/b8826fb04ce335da90dd4790f28790df?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/felisucoibi"
        },
        "author_username": "felisucoibi",
        "user_id": 29079
      },
      {
        "project_id": 1,
        "action_name": "opened",
        "target_id": 77640,
        "target_iid": 1331,
        "target_type": "Issue",
        "author_id": 29079,
        "target_title": "When searching for a file in a directory, if you delete the search term it closes the search bar",
        "created_at": "2020-01-01T18:31:08.853Z",
        "author": {
          "id": 29079,
          "name": "felisucoibi",
          "username": "felisucoibi",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/b8826fb04ce335da90dd4790f28790df?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/felisucoibi"
        },
        "author_username": "felisucoibi",
        "user_id": 29079
      },
      {
        "project_id": 1,
        "action_name": "commented on",
        "target_id": 676371,
        "target_iid": 676371,
        "target_type": "Note",
        "author_id": 29079,
        "target_title": "When searching for a file in a directory, if you delete the search term it closes the search bar",
        "created_at": "2020-01-03T10:07:58.280Z",
        "note": {
          "id": 676371,
          "type": null,
          "body": "I've never pressed control+f ever in my life, i use the mouse, some people thinks what they do is the good way... but there are other ways and GUI should be for everybody.\n\nIt's completely anti intuitive to delete a search term and that the search box dissapear, there is a button to make the search box to dissapear in the right of the search form so not needed to autodissapear if i delete search term.\n\nI reopen the issue.",
          "attachment": null,
          "author": {
            "id": 29079,
            "name": "felisucoibi",
            "username": "felisucoibi",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/b8826fb04ce335da90dd4790f28790df?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/felisucoibi"
          },
          "created_at": "2020-01-03T10:07:57.421Z",
          "updated_at": "2020-01-03T10:08:59.519Z",
          "system": false,
          "noteable_id": 77640,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 1331
        },
        "author": {
          "id": 29079,
          "name": "felisucoibi",
          "username": "felisucoibi",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/b8826fb04ce335da90dd4790f28790df?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/felisucoibi"
        },
        "author_username": "felisucoibi",
        "user_id": 29079
      },
      {
        "project_id": 1,
        "action_name": "opened",
        "target_id": 77640,
        "target_iid": 1331,
        "target_type": "Issue",
        "author_id": 29079,
        "target_title": "When searching for a file in a directory, if you delete the search term it closes the search bar",
        "created_at": "2020-01-03T10:07:58.315Z",
        "author": {
          "id": 29079,
          "name": "felisucoibi",
          "username": "felisucoibi",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/b8826fb04ce335da90dd4790f28790df?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/felisucoibi"
        },
        "author_username": "felisucoibi",
        "user_id": 29079
      },
      {
        "project_id": 1,
        "action_name": "commented on",
        "target_id": 676380,
        "target_iid": 676380,
        "target_type": "Note",
        "author_id": 29079,
        "target_title": "In search result left click and ADD an option to select all search results.",
        "created_at": "2020-01-03T10:12:29.074Z",
        "note": {
          "id": 676380,
          "type": null,
          "body": "Again somebody answer me saying there is a combinatio of keys i can use... and again i say it is a wrong approach, some people don't use keyboard shortcuts, surprise!! left click in a item and add select all is not something strange in ui design, is very common, i wonder wich kind of ui designers answers this way.... select all without using control+A is needed to select all the items.",
          "attachment": null,
          "author": {
            "id": 29079,
            "name": "felisucoibi",
            "username": "felisucoibi",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/b8826fb04ce335da90dd4790f28790df?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/felisucoibi"
          },
          "created_at": "2020-01-03T10:12:28.342Z",
          "updated_at": "2020-01-03T10:12:28.342Z",
          "system": false,
          "noteable_id": 77597,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 1330
        },
        "author": {
          "id": 29079,
          "name": "felisucoibi",
          "username": "felisucoibi",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/b8826fb04ce335da90dd4790f28790df?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/felisucoibi"
        },
        "author_username": "felisucoibi",
        "user_id": 29079
      },
      {
        "project_id": 1,
        "action_name": "opened",
        "target_id": 77597,
        "target_iid": 1330,
        "target_type": "Issue",
        "author_id": 29079,
        "target_title": "In search result left click and ADD an option to select all search results.",
        "created_at": "2020-01-03T10:12:29.198Z",
        "author": {
          "id": 29079,
          "name": "felisucoibi",
          "username": "felisucoibi",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/b8826fb04ce335da90dd4790f28790df?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/felisucoibi"
        },
        "author_username": "felisucoibi",
        "user_id": 29079
      }
    ]
  },
  {
    "id": 29080,
    "events": [
      {
        "project_id": 1854,
        "action_name": "opened",
        "target_id": 77614,
        "target_iid": 161,
        "target_type": "Issue",
        "author_id": 29080,
        "target_title": "benchmark graph lines do not plot up to 100% when changing number of samples",
        "created_at": "2019-12-31T20:38:01.121Z",
        "author": {
          "id": 29080,
          "name": "tethragon",
          "username": "tethragon",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/991b2ea5e6827e1d1efdc0f152972cd1?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/tethragon"
        },
        "author_username": "tethragon",
        "user_id": 29080
      }
    ]
  },
  {
    "id": 29081,
    "events": [
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 675478,
        "target_iid": 675478,
        "target_type": "Note",
        "author_id": 29081,
        "target_title": "GIMP 2.10.14 does not start on W10 (affects users using gimp-2.10.14-setup-2.exe on nVidia GPUs)",
        "created_at": "2019-12-31T16:38:18.983Z",
        "note": {
          "id": 675478,
          "type": null,
          "body": "Also experiencing this issue, switching to -1 installer worked.\n* OS: Windows 10 Pro (1809)\n* GPU: GTX 1080 (Game Ready Driver 441.66)\n* CPU: Ryzen TR 2950X\n* Anti-virus: Malwarebytes",
          "attachment": null,
          "author": {
            "id": 29081,
            "name": "Fakas",
            "username": "Fakas",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/7935b5b570dfcc7b59f699483e3dd740?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/Fakas"
          },
          "created_at": "2019-12-31T16:38:18.034Z",
          "updated_at": "2019-12-31T16:38:45.200Z",
          "system": false,
          "noteable_id": 77168,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4385
        },
        "author": {
          "id": 29081,
          "name": "Fakas",
          "username": "Fakas",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/7935b5b570dfcc7b59f699483e3dd740?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/Fakas"
        },
        "author_username": "Fakas",
        "user_id": 29081
      },
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 675682,
        "target_iid": 675682,
        "target_type": "Note",
        "author_id": 29081,
        "target_title": "GIMP 2.10.14 does not start on W10 (affects users using gimp-2.10.14-setup-2.exe on nVidia GPUs)",
        "created_at": "2020-01-01T15:23:03.610Z",
        "note": {
          "id": 675682,
          "type": null,
          "body": "@jernejs GIMP launches and appears to work properly on my system with both test1 and test2.",
          "attachment": null,
          "author": {
            "id": 29081,
            "name": "Fakas",
            "username": "Fakas",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/7935b5b570dfcc7b59f699483e3dd740?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/Fakas"
          },
          "created_at": "2020-01-01T15:23:02.316Z",
          "updated_at": "2020-01-01T15:23:02.316Z",
          "system": false,
          "noteable_id": 77168,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4385
        },
        "author": {
          "id": 29081,
          "name": "Fakas",
          "username": "Fakas",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/7935b5b570dfcc7b59f699483e3dd740?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/Fakas"
        },
        "author_username": "Fakas",
        "user_id": 29081
      }
    ]
  },
  {
    "id": 29082,
    "events": [
      {
        "project_id": 5895,
        "action_name": "opened",
        "target_id": 77602,
        "target_iid": 64,
        "target_type": "Issue",
        "author_id": 29082,
        "target_title": "Destroys home \"system\" folders with cyclic symlinks",
        "created_at": "2019-12-31T18:11:33.765Z",
        "author": {
          "id": 29082,
          "name": "frenchfaso",
          "username": "frenchfaso",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/0d23e9a2694fad9f344edf9d5b24272f?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/frenchfaso"
        },
        "author_username": "frenchfaso",
        "user_id": 29082
      }
    ]
  },
  {
    "id": 29083,
    "events": [
      {
        "project_id": 1906,
        "action_name": "opened",
        "target_id": 77604,
        "target_iid": 1032,
        "target_type": "Issue",
        "author_id": 29083,
        "target_title": "Provide help documentation for bookmarks",
        "created_at": "2019-12-31T18:44:07.035Z",
        "author": {
          "id": 29083,
          "name": "Alek Zikon",
          "username": "alekzikon",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/582ae7f1dcff755e2b2d6949f9d39620?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/alekzikon"
        },
        "author_username": "alekzikon",
        "user_id": 29083
      }
    ]
  },
  {
    "id": 29084,
    "events": [
      {
        "project_id": 1848,
        "action_name": "opened",
        "target_id": 77603,
        "target_iid": 4433,
        "target_type": "Issue",
        "author_id": 29084,
        "target_title": "GIMP segfaults",
        "created_at": "2019-12-31T18:15:56.097Z",
        "author": {
          "id": 29084,
          "name": "Annette Mei\u00dfner",
          "username": "meissner.annette",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/276bb4f5986ae729aa6365079059df6d?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/meissner.annette"
        },
        "author_username": "meissner.annette",
        "user_id": 29084
      }
    ]
  },
  {
    "id": 29085,
    "events": [
      {
        "project_id": 9712,
        "action_name": "created",
        "target_id": null,
        "target_iid": null,
        "target_type": null,
        "author_id": 29085,
        "target_title": null,
        "created_at": "2019-12-31T18:40:13.676Z",
        "author": {
          "id": 29085,
          "name": "Pieter van Oostrum",
          "username": "pietervo",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/df1429ae5a9139e89182d6b038534ee5?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/pietervo"
        },
        "author_username": "pietervo",
        "user_id": 29085
      },
      {
        "project_id": 9712,
        "action_name": "pushed new",
        "target_id": null,
        "target_iid": null,
        "target_type": null,
        "author_id": 29085,
        "target_title": null,
        "created_at": "2019-12-31T19:21:15.310Z",
        "author": {
          "id": 29085,
          "name": "Pieter van Oostrum",
          "username": "pietervo",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/df1429ae5a9139e89182d6b038534ee5?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/pietervo"
        },
        "push_data": {
          "commit_count": 0,
          "action": "created",
          "ref_type": "branch",
          "commit_from": null,
          "commit_to": "0815302dee2b78139832c2080348086a0564836b",
          "ref": "python-tests",
          "commit_title": null,
          "ref_count": null
        },
        "author_username": "pietervo",
        "user_id": 29085
      },
      {
        "project_id": 9712,
        "action_name": "pushed to",
        "target_id": null,
        "target_iid": null,
        "target_type": null,
        "author_id": 29085,
        "target_title": null,
        "created_at": "2019-12-31T20:51:27.553Z",
        "author": {
          "id": 29085,
          "name": "Pieter van Oostrum",
          "username": "pietervo",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/df1429ae5a9139e89182d6b038534ee5?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/pietervo"
        },
        "push_data": {
          "commit_count": 1,
          "action": "pushed",
          "ref_type": "branch",
          "commit_from": "0815302dee2b78139832c2080348086a0564836b",
          "commit_to": "4b0e6323e6faa284a1d09a16d8c1904c31fa5f09",
          "ref": "python-tests",
          "commit_title": "Updated python/tests/tstLastError.py",
          "ref_count": null
        },
        "author_username": "pietervo",
        "user_id": 29085
      },
      {
        "project_id": 1665,
        "action_name": "opened",
        "target_id": 21216,
        "target_iid": 64,
        "target_type": "MergeRequest",
        "author_id": 29085,
        "target_title": "Updated python/tests/tstLastError.py",
        "created_at": "2019-12-31T21:01:54.949Z",
        "author": {
          "id": 29085,
          "name": "Pieter van Oostrum",
          "username": "pietervo",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/df1429ae5a9139e89182d6b038534ee5?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/pietervo"
        },
        "author_username": "pietervo",
        "user_id": 29085
      },
      {
        "project_id": 9712,
        "action_name": "pushed new",
        "target_id": null,
        "target_iid": null,
        "target_type": null,
        "author_id": 29085,
        "target_title": null,
        "created_at": "2020-01-01T18:19:04.896Z",
        "author": {
          "id": 29085,
          "name": "Pieter van Oostrum",
          "username": "pietervo",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/df1429ae5a9139e89182d6b038534ee5?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/pietervo"
        },
        "push_data": {
          "commit_count": 1,
          "action": "created",
          "ref_type": "branch",
          "commit_from": null,
          "commit_to": "6a7dcb5edb17d73e5814589edfbf2a246c2ef456",
          "ref": "python-tests2",
          "commit_title": "Updated Python test reader2.py",
          "ref_count": null
        },
        "author_username": "pietervo",
        "user_id": 29085
      },
      {
        "project_id": 1665,
        "action_name": "opened",
        "target_id": 21227,
        "target_iid": 65,
        "target_type": "MergeRequest",
        "author_id": 29085,
        "target_title": "Updated Python test reader2.py",
        "created_at": "2020-01-01T18:30:06.116Z",
        "author": {
          "id": 29085,
          "name": "Pieter van Oostrum",
          "username": "pietervo",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/df1429ae5a9139e89182d6b038534ee5?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/pietervo"
        },
        "author_username": "pietervo",
        "user_id": 29085
      }
    ]
  },
  {
    "id": 29089,
    "events": [
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 675553,
        "target_iid": 675553,
        "target_type": "Note",
        "author_id": 29089,
        "target_title": "Opening png, jpeg or tiff with non-latin unicode path fails to load metadata",
        "created_at": "2019-12-31T20:10:05.345Z",
        "note": {
          "id": 675553,
          "type": "DiscussionNote",
          "body": "Open file-png.exe and if it says \"The code execution cannot proceed because **libgimpui-2.O-O.dll** was not found.\"\nIt's a typo because they should be **zeroes** not **Os**. How could someone fail at programming something simple? To avoid typing in **Os** instead of **zeroes**, use the keypad! **It's on the right side of the keyboard!** But, nope! That someone was lazy! There needs to be a keyboard that has the **zero** key **to the left** of the **one** key instead of **to the right** of the **nine** key. It needs to be intented.",
          "attachment": null,
          "author": {
            "id": 29089,
            "name": "David Royston",
            "username": "mcw4v3x",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/baa7ea4a69800ead4c8e39d6d33cc106?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/mcw4v3x"
          },
          "created_at": "2019-12-31T20:10:04.612Z",
          "updated_at": "2019-12-31T20:28:09.190Z",
          "system": false,
          "noteable_id": 40585,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 1350
        },
        "author": {
          "id": 29089,
          "name": "David Royston",
          "username": "mcw4v3x",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/baa7ea4a69800ead4c8e39d6d33cc106?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/mcw4v3x"
        },
        "author_username": "mcw4v3x",
        "user_id": 29089
      }
    ]
  },
  {
    "id": 29091,
    "events": []
  },
  {
    "id": 29093,
    "events": [
      {
        "project_id": 665,
        "action_name": "opened",
        "target_id": 78235,
        "target_iid": 2375,
        "target_type": "Issue",
        "author_id": 29093,
        "target_title": "Missing method \"gtk_accel_map_remove_entry\"",
        "created_at": "2020-01-14T10:38:45.688Z",
        "author": {
          "id": 29093,
          "name": "Alexander Schwinn",
          "username": "al.schwinn",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/3df1b41f9a5e05c68f5e846e7195cc5a?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/al.schwinn"
        },
        "author_username": "al.schwinn",
        "user_id": 29093
      }
    ]
  },
  {
    "id": 29094,
    "events": []
  },
  {
    "id": 29095,
    "events": [
      {
        "project_id": 3560,
        "action_name": "opened",
        "target_id": 77621,
        "target_iid": 882,
        "target_type": "Issue",
        "author_id": 29095,
        "target_title": "Epson stylus NX430 Scanner Hangs after scanning document",
        "created_at": "2019-12-31T23:52:39.549Z",
        "author": {
          "id": 29095,
          "name": "Mad-Professor",
          "username": "Mad-Professor",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/26e039f7f698bc8ee9fd1c504211b3ac?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/Mad-Professor"
        },
        "author_username": "Mad-Professor",
        "user_id": 29095
      },
      {
        "project_id": 3560,
        "action_name": "commented on",
        "target_id": 675843,
        "target_iid": 675843,
        "target_type": "Note",
        "author_id": 29095,
        "target_title": "Epson stylus NX430 Scanner Hangs after scanning document",
        "created_at": "2020-01-01T22:45:05.701Z",
        "note": {
          "id": 675843,
          "type": null,
          "body": "I've installed flatpak and paperwork but for some reason it installed 1.2.4 using this command\n\n`flatpak --user install https://builder.openpaper.work/paperwork_release.flatpakref `\n\nfor some reason and had to use \n\n`flatpak --user install https://builder.openpaper.work/paperwork_master.flatpakref`\n\nto get 1.3.1.\n\nI wasn't sure if I needed saned running, so I did it anyways and added the loop address to conf.\nSo far the scanner has worked perfectly using flatpak version.\n\nI've also flagged the packages on arch for update, so we should see a testing version of it soon.\n\nI'll try getting ironscanner working on my system since there's no package or aur build for it, I'll have to compile from source which might take some time on this old intel 2500k.\n\nEDIT: I compiled ironscanner and ran it, sent report.\nhttps://openpaper.work/en-us/scanner_db/report/419/",
          "attachment": null,
          "author": {
            "id": 29095,
            "name": "Mad-Professor",
            "username": "Mad-Professor",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/26e039f7f698bc8ee9fd1c504211b3ac?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/Mad-Professor"
          },
          "created_at": "2020-01-01T22:45:05.050Z",
          "updated_at": "2020-01-01T23:59:54.308Z",
          "system": false,
          "noteable_id": 77621,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 882
        },
        "author": {
          "id": 29095,
          "name": "Mad-Professor",
          "username": "Mad-Professor",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/26e039f7f698bc8ee9fd1c504211b3ac?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/Mad-Professor"
        },
        "author_username": "Mad-Professor",
        "user_id": 29095
      }
    ]
  },
  {
    "id": 29096,
    "events": [
      {
        "project_id": 1848,
        "action_name": "commented on",
        "target_id": 675600,
        "target_iid": 675600,
        "target_type": "Note",
        "author_id": 29096,
        "target_title": "GIMP 2.10.14 does not start on W10 (affects users using gimp-2.10.14-setup-2.exe on nVidia GPUs)",
        "created_at": "2020-01-01T00:34:28.324Z",
        "note": {
          "id": 675600,
          "type": null,
          "body": "Hello, \n  I had the same issue, 8 hour old computer (new build.) \nFresh install of Gimp wouldn't start. I replaced the DLLs with the openblas-test1.zip and Gimp now opens.\n* OS:Windows 10 Pro\n* GPU:MSI 1080 TI Gaming\n* CPU: Ryzen 3900X\n* Antivirus: Microsoft\n\nHope that helps.",
          "attachment": null,
          "author": {
            "id": 29096,
            "name": "Mike L",
            "username": "michiel.lemaster",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/927555fb97eeef75ebd9f8332f1255c9?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/michiel.lemaster"
          },
          "created_at": "2020-01-01T00:34:27.058Z",
          "updated_at": "2020-01-01T00:34:27.058Z",
          "system": false,
          "noteable_id": 77168,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 4385
        },
        "author": {
          "id": 29096,
          "name": "Mike L",
          "username": "michiel.lemaster",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/927555fb97eeef75ebd9f8332f1255c9?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/michiel.lemaster"
        },
        "author_username": "michiel.lemaster",
        "user_id": 29096
      }
    ]
  },
  {
    "id": 29097,
    "events": [
      {
        "project_id": 1848,
        "action_name": "opened",
        "target_id": 77623,
        "target_iid": 4435,
        "target_type": "Issue",
        "author_id": 29097,
        "target_title": "GIMP crashed with a fatal error: fatal error: Segmentation fault: 11",
        "created_at": "2020-01-01T02:42:19.864Z",
        "author": {
          "id": 29097,
          "name": "_call_me_ red_",
          "username": "bondilocal101",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/b2a6a2be0a35bade4424fb99d95e78a2?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/bondilocal101"
        },
        "author_username": "bondilocal101",
        "user_id": 29097
      }
    ]
  },
  {
    "id": 29098,
    "events": [
      {
        "project_id": 295,
        "action_name": "opened",
        "target_id": 77624,
        "target_iid": 137,
        "target_type": "Issue",
        "author_id": 29098,
        "target_title": "Crashes viewing DNG files",
        "created_at": "2020-01-01T02:50:55.472Z",
        "author": {
          "id": 29098,
          "name": "Rihards Balins",
          "username": "Richy91",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/681729984d41c8157131b23c3f99c108?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/Richy91"
        },
        "author_username": "Richy91",
        "user_id": 29098
      },
      {
        "project_id": 295,
        "action_name": "commented on",
        "target_id": 675740,
        "target_iid": 675740,
        "target_type": "Note",
        "author_id": 29098,
        "target_title": "Crashes viewing DNG files",
        "created_at": "2020-01-01T18:20:39.063Z",
        "note": {
          "id": 675740,
          "type": null,
          "body": "All i can get is this! \n\nThread 1 \"gnome-photos\" received signal SIGSEGV, Segmentation fault.\n0x00005555555a597a in photos_base_item_load_load_buffer (\n    source_object=<optimized out>, res=<optimized out>, \n    user_data=<optimized out>) at ../src/photos-base-item.c:1754\n1754\t../src/photos-base-item.c: No such file or directory.\n\n\n[dng-crash](/uploads/ad09787587b923e7439d310b98631ed2/dng-crash)",
          "attachment": null,
          "author": {
            "id": 29098,
            "name": "Rihards Balins",
            "username": "Richy91",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/681729984d41c8157131b23c3f99c108?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/Richy91"
          },
          "created_at": "2020-01-01T18:20:38.363Z",
          "updated_at": "2020-01-01T21:23:15.683Z",
          "system": false,
          "noteable_id": 77624,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 137
        },
        "author": {
          "id": 29098,
          "name": "Rihards Balins",
          "username": "Richy91",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/681729984d41c8157131b23c3f99c108?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/Richy91"
        },
        "author_username": "Richy91",
        "user_id": 29098
      },
      {
        "project_id": 295,
        "action_name": "commented on",
        "target_id": 675785,
        "target_iid": 675785,
        "target_type": "Note",
        "author_id": 29098,
        "target_title": "Opening XCF from GIMP 2.10 crashes the application.",
        "created_at": "2020-01-01T20:10:06.679Z",
        "note": {
          "id": 675785,
          "type": null,
          "body": "I tested this also as long as I am testing .dng image crash.\nI am using 3.34 version. And created simple .xcf file in gimp.\nI tested also .xbm, .xpm, .bmp file and got the same crash. (EOG opens fine)\n\nThat is all i get from crash + some gegl warnings (i can post them if needed)!\n\nThread 1 \"gnome-photos\" received signal SIGSEGV, Segmentation fault.\n0x00005555555a597a in ?? ()",
          "attachment": null,
          "author": {
            "id": 29098,
            "name": "Rihards Balins",
            "username": "Richy91",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/681729984d41c8157131b23c3f99c108?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/Richy91"
          },
          "created_at": "2020-01-01T20:10:05.987Z",
          "updated_at": "2020-01-01T20:10:05.987Z",
          "system": false,
          "noteable_id": 74960,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 134
        },
        "author": {
          "id": 29098,
          "name": "Rihards Balins",
          "username": "Richy91",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/681729984d41c8157131b23c3f99c108?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/Richy91"
        },
        "author_username": "Richy91",
        "user_id": 29098
      },
      {
        "project_id": 295,
        "action_name": "commented on",
        "target_id": 675804,
        "target_iid": 675804,
        "target_type": "Note",
        "author_id": 29098,
        "target_title": "Opening XCF from GIMP 2.10 crashes the application.",
        "created_at": "2020-01-01T21:18:54.249Z",
        "note": {
          "id": 675804,
          "type": null,
          "body": "I guess this should be it! Let me know if it is right!\n\n[trace](/uploads/5cfaddbc3afc1f16144001290bb7a17d/trace)",
          "attachment": null,
          "author": {
            "id": 29098,
            "name": "Rihards Balins",
            "username": "Richy91",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/681729984d41c8157131b23c3f99c108?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/Richy91"
          },
          "created_at": "2020-01-01T21:18:53.620Z",
          "updated_at": "2020-01-01T21:18:53.620Z",
          "system": false,
          "noteable_id": 74960,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 134
        },
        "author": {
          "id": 29098,
          "name": "Rihards Balins",
          "username": "Richy91",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/681729984d41c8157131b23c3f99c108?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/Richy91"
        },
        "author_username": "Richy91",
        "user_id": 29098
      },
      {
        "project_id": 1501,
        "action_name": "opened",
        "target_id": 77957,
        "target_iid": 255,
        "target_type": "Issue",
        "author_id": 29098,
        "target_title": "Adwaita dark: Cannot read highlighted line due to lack of contrast; grid pattern makes everything unreadable",
        "created_at": "2020-01-07T20:06:35.490Z",
        "author": {
          "id": 29098,
          "name": "Rihards Balins",
          "username": "Richy91",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/681729984d41c8157131b23c3f99c108?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/Richy91"
        },
        "author_username": "Richy91",
        "user_id": 29098
      },
      {
        "project_id": 1501,
        "action_name": "commented on",
        "target_id": 680831,
        "target_iid": 680831,
        "target_type": "Note",
        "author_id": 29098,
        "target_title": "Adwaita dark: Cannot read highlighted line due to lack of contrast; grid pattern makes everything unreadable",
        "created_at": "2020-01-08T23:45:38.464Z",
        "note": {
          "id": 680831,
          "type": null,
          "body": "You can't read words when search highlights text.\n\n![highlight](/uploads/c440fe6f7ade47a6fcabf00657877b87/highlight.png)",
          "attachment": null,
          "author": {
            "id": 29098,
            "name": "Rihards Balins",
            "username": "Richy91",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/681729984d41c8157131b23c3f99c108?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/Richy91"
          },
          "created_at": "2020-01-08T23:45:37.984Z",
          "updated_at": "2020-01-08T23:45:37.984Z",
          "system": false,
          "noteable_id": 77957,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 255
        },
        "author": {
          "id": 29098,
          "name": "Rihards Balins",
          "username": "Richy91",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/681729984d41c8157131b23c3f99c108?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/Richy91"
        },
        "author_username": "Richy91",
        "user_id": 29098
      }
    ]
  },
  {
    "id": 29099,
    "events": [
      {
        "project_id": 142,
        "action_name": "opened",
        "target_id": 77625,
        "target_iid": 165,
        "target_type": "Issue",
        "author_id": 29099,
        "target_title": "On 1st Jan all podcasts from previous year are listed under \"Older\"",
        "created_at": "2020-01-01T04:28:23.953Z",
        "author": {
          "id": 29099,
          "name": "Martin Rixham",
          "username": "MartinRixham",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/08b9136e9857e08a4d741222dc7e7cd6?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/MartinRixham"
        },
        "author_username": "MartinRixham",
        "user_id": 29099
      }
    ]
  },
  {
    "id": 29100,
    "events": []
  },
  {
    "id": 29102,
    "events": []
  },
  {
    "id": 29105,
    "events": [
      {
        "project_id": 1,
        "action_name": "commented on",
        "target_id": 675645,
        "target_iid": 675645,
        "target_type": "Note",
        "author_id": 29105,
        "target_title": "Improve running binaries experience",
        "created_at": "2020-01-01T12:36:50.056Z",
        "note": {
          "id": 675645,
          "type": null,
          "body": "Could we, at least, change the default behavior to `ask` (see @pablo.bianchi comment)? The user currently has to jump through hoops to run scripts, which is insane in terms of UX (from Ubuntu 19.10).",
          "attachment": null,
          "author": {
            "id": 29105,
            "name": "Charles Grassin",
            "username": "CGrassin",
            "state": "active",
            "avatar_url": "https://secure.gravatar.com/avatar/8d52275b7e2255ab359e20fa72660163?s=80&d=identicon",
            "web_url": "https://gitlab.gnome.org/CGrassin"
          },
          "created_at": "2020-01-01T12:36:48.497Z",
          "updated_at": "2020-01-01T12:36:48.497Z",
          "system": false,
          "noteable_id": 18753,
          "noteable_type": "Issue",
          "resolvable": false,
          "noteable_iid": 443
        },
        "author": {
          "id": 29105,
          "name": "Charles Grassin",
          "username": "CGrassin",
          "state": "active",
          "avatar_url": "https://secure.gravatar.com/avatar/8d52275b7e2255ab359e20fa72660163?s=80&d=identicon",
          "web_url": "https://gitlab.gnome.org/CGrassin"
        },
        "author_username": "CGrassin",
        "user_id": 29105
      }
    ]
  }
]